<!DOCTYPE html>
<html  >
<head>
  
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  
  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
  <link rel="shortcut icon" href="assets/images/ahc2-human-145x145.png" type="image/x-icon">
  <meta name="description" content="">
  
  
  <title>Abstracts</title>
  <link rel="stylesheet" href="assets/web/assets/mobirise-icons2/mobirise2.css">
  <link rel="stylesheet" href="assets/bootstrap/css/bootstrap.min.css">
  <link rel="stylesheet" href="assets/bootstrap/css/bootstrap-grid.min.css">
  <link rel="stylesheet" href="assets/bootstrap/css/bootstrap-reboot.min.css">
  <link rel="stylesheet" href="assets/parallax/jarallax.css">
  <link rel="stylesheet" href="assets/web/assets/gdpr-plugin/gdpr-styles.css">
  <link rel="stylesheet" href="assets/animatecss/animate.css">
  <link rel="stylesheet" href="assets/dropdown/css/style.css">
  <link rel="stylesheet" href="assets/socicon/css/styles.css">
  <link rel="stylesheet" href="assets/theme/css/style.css">
  <link rel="preload" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i&display=swap" as="style" onload="this.onload=null;this.rel='stylesheet'">
  <noscript><link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i&display=swap"></noscript>
  <link rel="preload" as="style" href="assets/mobirise/css/mbr-additional.css"><link rel="stylesheet" href="assets/mobirise/css/mbr-additional.css" type="text/css">

  
  
  
</head>
<body>
  
  <section data-bs-version="5.1" class="menu menu1 cid-sUvgvfOreC" once="menu" id="menu1-2x">
    

    <nav class="navbar navbar-dropdown navbar-fixed-top navbar-expand-lg">
        <div class="container">
            <div class="navbar-brand">
                <span class="navbar-logo">
                    <a href="index.html#top">
                        <img src="assets/images/ahc2-human-145x145.png" alt="AHC" style="height: 4.5rem;">
                    </a>
                </span>
                <span class="navbar-caption-wrap"><a class="navbar-caption text-black text-primary display-7" href="index.html">Applied HoloGenomics Conference 2024&nbsp;<br>30th June - 2nd July<br>Copenhagen, Denmark&nbsp;#AHC2024</a></span>
            </div>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbarSupportedContent" data-bs-target="#navbarSupportedContent" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
                <div class="hamburger">
                    <span></span>
                    <span></span>
                    <span></span>
                    <span></span>
                </div>
            </button>
            <div class="collapse navbar-collapse" id="navbarSupportedContent">
                <ul class="navbar-nav nav-dropdown nav-right" data-app-modern-menu="true"><li class="nav-item dropdown"><a class="nav-link link text-black dropdown-toggle display-7" href="#" data-toggle="dropdown-submenu" data-bs-toggle="dropdown" data-bs-auto-close="outside" aria-expanded="false">Conference Scope</a><div class="dropdown-menu" aria-labelledby="dropdown-undefined"><a class="text-black dropdown-item text-primary display-7" href="index.html#content4-1f">Conference Scope</a><a class="text-black dropdown-item text-primary display-7" href="index.html#gallery3-q" aria-expanded="false">Key Themes</a><a class="text-black show dropdown-item text-primary display-7" href="index.html#slider6-7w">Invited Speakers</a></div></li>
                    <li class="nav-item dropdown"><a class="nav-link link text-black text-primary dropdown-toggle display-7" href="agenda_tba.html" data-toggle="dropdown-submenu" data-bs-toggle="dropdown" data-bs-auto-close="outside" aria-expanded="false">Programme</a><div class="dropdown-menu" aria-labelledby="dropdown-889"><a class="text-black text-primary dropdown-item display-7" href="agenda.html">Programme<br></a><a class="text-black text-primary dropdown-item display-7" href="sessions.html">Planned Sessions<br></a><a class="text-black text-primary dropdown-item display-7" href="abstracts.html">Abstracts</a></div></li><li class="nav-item dropdown"><a class="nav-link link text-black dropdown-toggle display-7" href="#" data-toggle="dropdown-submenu" data-bs-toggle="dropdown" data-bs-auto-close="outside" aria-expanded="false">Useful Information</a><div class="dropdown-menu" aria-labelledby="dropdown-undefined"><a class="text-black dropdown-item text-primary display-7" href="index.html#content16-1j">Abstract and registration information</a><a class="text-black show dropdown-item text-primary display-7" href="index.html#content17-n">Useful Info</a><a class="text-black show dropdown-item text-primary display-7" href="index.html#features1-bp">Download Conference Material</a><a class="text-black show dropdown-item text-primary display-7" href="index.html#map1-o">Conference Location</a><a class="text-black show dropdown-item text-primary display-7" href="index.html#image1-9y">Acknowledgements</a><a class="text-black show dropdown-item text-primary display-7" href="AHC_conference_policies.html">Code of Conduct, Health &amp; Safety and Conference Policies</a><a class="text-black show dropdown-item text-primary display-7" href="CookiePolicy.html">Data / Privacy / Cookie Policies</a><a class="text-black show dropdown-item text-primary display-7" href="index.html#footer4-k">Contact</a></div></li><li class="nav-item"><a class="nav-link link text-black text-primary display-7" href="news.html">News</a></li></ul>
                <div class="icons-menu">
                    <a class="iconfont-wrapper" href="https://twitter.com/AppliedHoloConf" target="_blank">
                        <span class="p-2 mbr-iconfont socicon-twitter socicon" style="color: rgb(177, 95, 98); fill: rgb(177, 95, 98);"></span>
                    </a>
                    
                    
                    
                </div>
                
            </div>
        </div>
    </nav>
</section>

<section data-bs-version="5.1" class="header9 cid-tgdLZpbp4V" id="header9-37">

    

    

    <div class="text-center container">
        <div class="row justify-content-center">
            <div class="col-md-12 col-lg-9">
                
                <h1 class="mbr-section-title mbr-fonts-style mb-3 display-2"><strong>Conference Abstracts</strong></h1>
                
                <div class="mbr-section-btn mt-3"><a class="btn btn-primary display-4" href="AHC_Abstracts.html#content4-38">Opening Day</a>
                    <a class="btn btn-primary display-4" href="AHC_Abstracts.html#content4-39">Session 1: Human Health</a> <a class="btn btn-primary display-4" href="AHC_Abstracts.html#content4-3a">Session 2: Agrisciences and Sustainability&nbsp;</a> <a class="btn btn-primary display-4" href="AHC_Abstracts.html#content4-40">Session 3: Biodiversity and Conservation</a> <a class="btn btn-primary display-4" href="AHC_Abstracts.html#content4-45">Session 4: Future Technical Trends</a>  <a class="btn btn-primary display-4" href="AHC_Abstracts.html#content4-bc">Posters</a></div>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content4 cid-th71DXZOsY mbr-parallax-background" id="content4-38">
    
    <div class="mbr-overlay" style="opacity: 0.4; background-color: rgb(233, 209, 210);">
    </div>
    <div class="container">
        <div class="row justify-content-center">
            <div class="title col-md-12 col-lg-7">
                <h3 class="mbr-section-title mbr-fonts-style align-center mb-4 display-1"><strong>Opening Day</strong></h3>
                
                
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th8nWoaY3T" id="content10-4b">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Introductory Keynote: </strong></div><br><div><strong>The Holobiont Imperative – Why We Have to Rethink (Our) Nature</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Thomas Bosch [Kiel Life Science Center]</strong><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Animals have evolved within the framework of the microbes. The fact that most if not all animals have been colonized by microbes since the beginning of evolutionary history must lead to a radical rethinking of what an organism is, how it functions, and how it evolved. We need to rethink the origin and function of the immune system as well as the nervous system. The new discoveries must also prompt us think about our lifestyle and an environment that allows us to age in good health.  Language terms also need to be reconsidered as does the question of how we now define our "self". What exciting times in which the Applied Hologenomics Conference is taking place.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th8oaspeP5" id="content10-4c">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Moderated discussion with audience / Q&amp;A:</strong><br><strong><br></strong><div><strong>"Life, the Holobiont, and Everything"</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><div><strong>Assoc. Prof. Antton Alberdi [University of Copenhagen] (Moderator)</strong></div><div><strong>Prof. Thomas Bosch [Kiel Life Science Center]</strong></div><div><strong>Prof. Monica Medina [Pennsylvania State University]&nbsp;</strong><br><br></div></h6>
            </div>
            <div class="col-md-12 col-lg-4">
                <p class="mbr-text mbr-fonts-style display-7"></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content12 cid-th8oUVF0t9" id="content12-4e">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-md-12 col-lg-10">
                <div class="mbr-section-btn align-center"><a class="btn btn-primary display-4" href="AHC_Abstracts.html#top"><span class="mobi-mbri mobi-mbri-arrow-up mbr-iconfont mbr-iconfont-btn"></span>Back to top</a></div>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content4 cid-th71GFgWaH" id="content4-39">
    
    <div class="mbr-overlay" style="opacity: 0.7; background-color: rgb(233, 209, 210);">
    </div>
    <div class="container">
        <div class="row justify-content-center">
            <div class="title col-md-12 col-lg-10">
                <h3 class="mbr-section-title mbr-fonts-style align-center mb-4 display-1">
                    <strong>Session 1: Human Health</strong></h3>
                
                
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pcHVZF2" id="content10-3i">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Devices and Materials for Sampling/Manipulating the Human Microbiome</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Anja Boijsen [Technical University of Denmark]</strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Micro and nano engineering can be applied to realize delivery vehicles for e.g. probiotics as well as Ingestible devices for sampling gut content. We will show examples of encapsulation and delivery of bacteria using multilayered electrospun constructs. Here, the <em>in vivo </em>studies are performed in rats. Next, we will show the potential of micro as well as cm sized devices for sampling bacteria in the stomach/intestine. Here we have tested a range of animals (e.g. rats and pigs), with a variety of device designs. Initial results as well as future ideas will be presented.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pGzXrte" id="content10-3j">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Plasmid Dynamics and Maturation of the Infant Gut Microbiome</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Søren Sørensen [University of Copenhagen]</strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The escalating global challenge of antimicrobial resistance (AMR) alongside the enigmatic role of plasmids within the human gut microbiome constitutes a frontier for microbial ecology and public health research. This presentation synthesizes findings from two studies conducted by my group, shedding light on the intricacies of AMR and plasmid dynamics within the gut microbiomes of early life. The first study unravels the acquisition and determinants of antibiotic resistance genes (ARGs) in a cohort of 662 Danish children during their first year, revealing a bimodal distribution of ARG richness influenced predominantly by gut microbiome composition, particularly <em>E. coli</em>. Notably, environmental factors including antibiotic exposure play a significant role in shaping ARG profiles, which in turn correlate with gut microbiome maturity and potential health risks such as asthma. The second study pioneers a novel approach to plasmid analysis in the gut microbiomes of 34 mother-child cohorts, uncovering a previously underestimated plasmid diversity. This exploration not only expands our understanding of plasmid-host interactions but also illuminates the mechanisms through which plasmids enhance bacterial adaptability, especially in infants. Together, these studies offer novel insights into the microbial dynamics of the gut, emphasizing the critical need for innovative strategies to manage AMR and understand microbial gene transfer mechanisms in the context of human health.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pHqw5Xi" id="content10-3k">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Metabolic Microbiome-Host Interactions and its Role in Inflammatory Bowel Disease</strong><br></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Jan Taubenheim [University Kiel/University Hospital Kiel]</strong><br><br><em>Coauthors: Samer Kadib Alban, Johannes Zimmermann, Christoph Kaleta</em><div><br></div></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Inflammatory bowel disease (IBD) is characterized by chronic gastric inflammation with unknown causes, leading to significantly reduced life quality and lifespan. However, changes in the microbial composition, the intestinal, and the systemic metabolism are associated with an inflammatory response likely contributing to the onset and progression of IBD. We studied host and microbial metabolic interactions by using 16S-sequencing, transcriptomic, and metabolomics data to reconstruct and analyze metabolic models. We associated alterations in metabolism with disease activity and identified inflammation-relevant changes in NAD, amino acid, one-carbon, and phospholipid metabolism – all highly interconnected. Notably, increased tryptophan degradation led to reduced serum tryptophan levels and impaired NAD de novo synthesis. Reduced activity in glutamate transamination reactions disrupted NH4 homeostasis, polyamine, and glutathione synthesis. Additionally, the one-carbon cycle was reduced, resulting in shifts in the phospholipid levels due to the depletion of choline as a methyl donor during the recovery of the cycle. Metabolic alterations in the bacterial community contributed to these changes to various degrees, underlining the importance of the host-microbe metabolic crosstalk in IBD progression/pathogenesis. Finally, we modeled dietary interventions to shift microbial metabolism toward a healthy state, which could be explored as new dietary intervention strategies.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pHVWCiR" id="content10-3l">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>SFB: Screening Metagenomic Data for an Ancient Human Symbiont</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><div><strong>Asst. Prof. Luisa W. Hugerth [Uppsala University]</strong><br></div><div><br><em>Coauthor: Emiko Yoshida&nbsp;</em></div></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Segmented filamentous bacteria, SFB, or <em>Candidatus Arthromitus spp.</em> are key players regulating the mammalian immune and nervous system development. SFB are spore-forming Clostridiales with small genomes, several auxotrophies and extremely low oxygen tolerance. They attach deeply to the small intestine and are thought to propagate vertically, mother to offspring. Cross-colonization experiments between mouse and rat SFB have shown that they cannot colonize outside their host species, reinforcing the tight co-evolution with their host. We have previously described the only known genome from a human-associated SFB. Here we have screened previously published 16S rRNA data from human and non-human primate infants for SFB. While non-human SFB follow the phylogeny of their hosts, humans display 16S tags closely related to human, dog and mouse SFB, begging the question of whether human hosts can accept other SFB lineages. From all available SFB genomes (four host species), we developed probes for re-assessing the SFB content of shotgun metagenomic data. By deepening our understanding of the ecology and co-evolution of SFB and humans, we may gain insight into early-life immune priming, inflammatory bowel disease, and more.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pIsUFqM" id="content10-3m">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Gut Microbiome Plasticity and Human Adaptability under Ecological Change</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Assoc. Prof Rachel Carmody [Harvard University]</strong><br><strong><br></strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">&nbsp;The human gut microbiome responds rapidly to dietary change and other ecological perturbation. Plasticity in the gut microbiome is a double-edged sword, potentially lending humans some extra-genomic capacity for adaptation to ecological change while also creating new opportunities for conflict if the gut microbiome departs from profiles to which our physiology has adapted. In this talk, I'll discuss recent work probing the benefits and costs of gut microbial plasticity for human health and highlight key opportunities for future research.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pJ6Gh80" id="content10-3n">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>The Hologenome Beyond Biology - an Arctic Indigenous Perspective on the Holobiont</strong><br></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Assoc. Prof. Aviaja Lyberth Hauptmann [University of Greenland]</strong><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Our approach to understanding biology has expanded with the holobiont "to embrace a holistic lexicon, theory, and concept”. With the holobiome it has become scientifically sound to see the interconnectedness between the body and the mind. The holobiome can only be understood through a paradigm of interconnectedness that challenges the reductionist paradigms that defined early research in human-microbe relations. Yet, we often continue to limit the scope of interconnectedness to the biosciences and the physical manifestations of the holobiont. Inspired by an Indigenous research framework we can be challenged to think of the holobiont as interconnected on a much broader scale, concretely interconnected to environment, communities, history, and culture. This understanding is important in particular contexts, specifically when working with Indigenous peoples, who are increasingly targeted as a treasure trove of holobiome diversity.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3GGefkNF" id="content10-au">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Saliva Shotgun Metagenomics Reveal Insights into Diversity and Function of Oral Microbiome in Pregnancy</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Anusha Thakkutte Antony [Uppsala University]</strong><br><br><em>Coauthors: Nagihan Bostanci, Angelika Silbereisen, Tina Esmaili, Maria C. Krog, Irene Sterpu, Zahra Bashir, Lars Engstrand, Eva Wiberg-Itzel, Henriette Svarre Nielsen, Luisa W. Hugerth, Ina Schuppe-Koistinen</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The microbiome plays a crucial role in pregnancy outcomes, including preterm birth, fetal growth, and immune development. At the same time, the microbiome must adapt to changing conditions during pregnancy, including altered mucosal structure in response to hormones, altered eating patterns, and decreased gut transit time. While this has been extensively explored in fecal samples, little is known about the oral cavity. Here we assessed the salivary microbiome of 42 healthy pregnant women and 73 women with a regular menstrual cycle, revealing several differences in oral microbiome between the two groups. The beta-diversity analysis found that pregnancy status accounts for 8.2% of variance in oral bacteria abundance, primarily due to lower abundances of <em>Leptotrichia spp, Neisseria spp.</em>, and <em>Selenomonas spp</em>. Additionally, groups were examined using Socransky colour complexes, and we discovered that the highly inflammatory red complex group had higher levels among pregnant women, with a lower total abundance of the green group. Differential abundance analysis revealed that 77 taxa were significantly different between the samples. Furthermore, the functional pathways of the bacterial populations in saliva were annotated using the KEGG database, and 55 functional modules differed according to pregnancy status. We observed bi-directional interactions, where the host’s hormonal signals influence oral microbial behaviour. Conversely, microbial metabolites can impact host health. In conclusion, pregnant women have a lower oral microbial diversity and richness compared to non-pregnant women, indicating that pregnancy significantly impacts the oral microbiome. Future studies should focus on maintaining dental health and minimizing dysbiosis during pregnancy.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3GGO9kdx" id="content10-av">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Building the World’s Largest Plasmid-Host and Phage-Host Interaction Atlas Using Proximity Ligation Sequencing</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Ivan Liachko [Phase Genomics]</strong><br><br><em>Coauthors: Benjamin Auch, Jonas Grove, Sam Bryson, Demi Glidden, Emily Reister, Stephen Eacker</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Plasmids are mediators of horizontal gene transfer (HGT) and serve as both a reservoir and vector for the spread of antimicrobial resistance (AMR) and virulence genes. Phages also shape the global ecosystem through their impacts on community composition and HGT. However, linking mobile elements to their microbial hosts has proven challenging without culture-based experiments. These experiments inherently require that all microbial hosts are culturable, typically restricting the diversity that can be surveyed and limiting our understanding of valuable mobilome relationships. Metagenomic proximity ligation sequencing is a powerful method for associating viruses and plasmids with their hosts directly in native microbial communities. It captures, <em>in vivo</em>, physical interactions between microbial genomes and the genetic material lytic and lysogenic phages, plasmids, and AMR genes. Like culturing experiments, these linkages offer direct evidence that mobile elements are present within a host cell, thereby establishing a mobile element-host pair. However, unlike culturing experiments, this approach does not require the isolation of living bacterial cells. In addition to binning microbial genomes, this technology allows us to simultaneously deconvolve viral and plasmid genomes, directly from metagenomes, and reconstruct interactions without culturing. Using large-scale application of this technology to samples in healthcare settings, wastewater, agricultural, and environmental contexts, we have generated the world’s largest repository of genomic data for phages, plasmid, and resistance elements connected with their host microbes.  We will discuss both published and unpublished work on the application of this technology to microbial modulation in healthcare settings, and effort to address the global AMR crisis.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3GHxgdiO" id="content10-aw">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>From Starting Cultures to Gut Health: The Importance of Lactic Acid Bacteria in the Life of Mongolian Pastoralists</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Alexander Hübner [Max Planck Institute for Evolutionary Anthropology]</strong><br><br><em>Coauthors: Yi Jia Liow, Lkhamsuren Khulan, Nyambal Tuul, Raphaela Stahl, Shannon R. West, Zandra S. Fagernäs, Sabri Bromage, Björn Reichhardt, Matthäus Rest, Paul A. Lawson, Soninkhishig Tsolmon, Christina G. Warinner</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">In Mongolia, consuming nutrient-dense animal milk has a 5,000-year long tradition, and it dominates the diet of Mongolian nomadic pastoralists even today. Its main carbohydrate, lactose, is a disaccharide that requires the enzyme lactase for digestion, but the genetic variants necessary for persistently expressing lactase into adulthood are mostly absent in Mongolia. This imposes a high risk of lactose intolerance, whose symptoms include bloating or diarrhoea, on the pastoralists. Cultural techniques to reduce the lactose content of dairy products includes fermentation using starting cultures that are enriched in lactic acid bacteria (LAB), and these starter cultures, known as khöröngo, are passed down from generation to generation as an heirloom.<br>To investigate whether LAB contribute to gut health as probiotics, we collected a large variety of dairy products alongside stool samples of the pastoralists regularly consuming these dairy products. We reconstructed more than 300 LAB genomes, including many previously undescribed species, from both the dairy products and the stool samples and could show a large overlap in strains, indicating that dairy products have been the likely source of the LAB in the gut. Together, our work highlights the integrative importance of LAB on the everyday life of Mongolian pastoralists.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content12 cid-th8p6w2wvW" id="content12-4f">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-md-12 col-lg-10">
                <div class="mbr-section-btn align-center"><a class="btn btn-primary display-4" href="AHC_Abstracts.html#top"><span class="mobi-mbri mobi-mbri-arrow-up mbr-iconfont mbr-iconfont-btn"></span>Back to top</a></div>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content4 cid-th71HvaDWr" id="content4-3a">
    
    <div class="mbr-overlay" style="opacity: 0.7; background-color: rgb(233, 209, 210);">
    </div>
    <div class="container">
        <div class="row justify-content-center">
            <div class="title col-md-12 col-lg-12">
                <h3 class="mbr-section-title mbr-fonts-style align-center mb-4 display-1"><strong>Session 2: Agrisciences and Sustainability</strong></h3>
                
                
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pMTvpFM" id="content10-3o">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Engineering Algal Microbiomes for Sustainable Food and Feed Production</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Lone Gram [Technical University of Denmark]</strong><br><br><em>Coauthors:&nbsp;Line Roager, Dóra Smahajcsik, Matilde Carujo Mira Emidio de Almeida, Shengda Zhang&nbsp;</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Microalgae are used as feed for live feed in several larviculture productions, and their associate microbiome, the phycosphere, influences the growth of the algae and potentially the subsequent pathogen control in fish larvae. Little is known about how the microalgal microbiome assembles and interacts with the host and how the microbiome interacts with potential pathogens. We have found that the algal host (Isochrysis, Tetraselmis or Conticribra) is a strong determining factor in the microbiome assembly and that the algal host may affect the function (measured as transcriptome) of the algal associated bacteria (Maribacter and Yoonia). Individual bacteria can antagonize fish pathogens, i.e. have potential probiotic effects, however, the interaction between the pathogen and a more complex microbiome remains unknown. We have developed an assay, using a gfp-tagged fish pathogen to demonstrate how microbiomes from different algal hosts and other aquatic organisms can be extracted and tested for antagonism against the pathogen, thus enabling development of a minimal inhibitory microbiome. Concludingly, algal microbiomes interact with their host and can potentially be engineered in a direction enabling better algal growth and suppression of relevant bacterial pathogens.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pO1fCLJ" id="content10-3p">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Rewilding Plant Microbiomes</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Jos Raaijmakers [Netherlands Institute of Ecology]</strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Over the past decade, research has shown that microorganisms living on and inside eukaryotes – the microbiota - are drivers of host health. For plants, microbiota can greatly expand their genomic capabilities by enhancing immunity, nutrient acquisition, and tolerance to environmental stresses. More than ever, plant microbiota are being considered as a lever to increase the sustainability of food production under climate change conditions. Emerging from this global interest to harness the largely unexplored functional potential of microbiota, the microbiome rewilding hypothesis posits that plant and animal health can be improved by reinstating key members of the diverse (ancestral) microbiota that were depleted or even lost through domestication and industrialization, including changes in diet, breeding and the (over)use of antibiotics, pesticides and fertilizers. In my talk, I will go ‘Back to the Roots’ of wild relatives of crop plants grown in their center of origin and describe how domestication affected the taxonomic and functional diversity of the microbiome and the reciprocal effects on stress resilience of modern crop cultivars.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pODtnPh" id="content10-3q">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Trait-Based Study Predicts Glycerol/Diol Dehydrates as a Key Function of the Gut Microbiota of Hindgut-Fermenting Carnivores</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Clarissa Schwab [Aarhus University]</strong><br><br><em>Coauthors: Qing Li, Hans-Joachim Ruscheweyh, Lærke Hartmann Østergaard, Micael Libertella, Kim Skalborg Simonsen, Shinichi Sunagawa, Alberto Scoma</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Today, most mammals are herbivores whose gastrointestinal tract is characterized by enlarged fore- or hindguts. These ‘fermentation vessels’ house complex microbial communities that contribute to the utilization of dietary material. To determine how a microbial multipurpose functional trait relates to gut environmental conditions, we collected fecal samples from captive wild animals (n=55) and determined occurrence and diversity of pdu and cob-cbi-hem gene clusters that mediate the transformation of diet-derived glycerol and 1,2-propanediol, and encode enzymes for cobalamin synthesis, a shared good of microbial communities. We combined metagenomics with metabolite and gene biomarker quantification and <em>in vitro </em>fermentation. Fecal levels of the glycerol transformation product 1,3-propanediol were higher in hindgut- than foregut fermenters. Gene analyses indicated that pduC harboring taxa are common feature that occurred more frequently in hindgut-fermenters. Phylogenetic analysis of MAGs identified captive wild animal fecal microbiota as taxonomically rich with a total of 4150 species but pointed at only 56 species that at least partially harbored pdu and cbi-cob-hem. While taxonomic diversity was highest in fecal samples of foregut fermenting herbivores, higher abundance and diversity of pdu/cbi-cob-hem related to higher potential for glycerol and 1,2-PD utilization of the less diverse microbiota of hindgut fermenting carnivores <em>in vitro</em>. Together, our approach stratified the potential of fecal microbiota for glycerol/1,2-propanediol utilization and cobalamin synthesis depending on host diet and gut physiology and suggests that the ability to utilize glycerol/1,2-propanediol is a key function of hindgut-fermenting carnivores, which links to the potential for cobalamin formation with potential relevance for microbiota and host.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pPdcSwl" id="content10-3r">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Enterosignatures of the Pig Gut Microbiota</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Ioanna-Theoni Vourlaki [Institute of Agrifood Research and Technology]</strong><br><br><em>Coauthors: Raquel Rio, Adria Clavell, Lino C. Ramírez-Ayala, Maria Ballester, Juan Pablo Sanchez, Miriam Piles, Raquel Quintanilla, Angela Cristina da Fonseca de Oliveira, Leandro Batista Costa, Antoni Dalmau, Yuliaxis Ramayo Caldas</em><strong><br></strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The pig gut microbiota is a complex ecosystem composed of microbial guilds, which until recently were unknown. Here, we decomposed the pig fecal microbiota of 648 healthy pigs during the transition (n=400) and growing-finishing (n=248) periods in co-occurring bacterial communities defined as pig enterosignatures (ES). Our results indicate that fecal microbiota can accurately be described by combinations of at least six ES, driven by Prevotella (ES-Prev), Treponema (ES-Trep), Lactobacillus (ES-Lact), Clostridium (ES-Clost), Streptococcus (ES-Strep), and UBA2810 (ES-UBA2). Enterosignatures assembly changes dynamically with the age of the host, where ES-Prev, ES-Strep, and ES-Lact seem to be core components, while ES-Trep, ES-Clost, and ES-UBA play age-specific roles. Our results suggest partial genetic control by the host, with heritabilities ranging from 0.30 to 0.42. Furthermore, our findings indicate that stress can impact ES assembly, decreasing ES-Lact abundance while increasing ES-Strep. We noted a positive link between ES-Prev and growth at 60-days, which later evolved negatively, impacting feed efficiency during the growing period. Additionally, a negative association of ES-Strep and a positive impact of ES-Lact on growth and feed efficiency at 190-days was observed. Remarkably, a negative association between the abundance of ES-Lact and levels of hair cortisol was also found during this period. Overall, our findings offer novel insights into the pig gut microbiota assembly, suggesting the usefulness of the ES in breeding and precision farming, but also underscoring the relevance of customizing microbial consortia based on the nutritional and health requirements at each stage of the porcine production chain.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pQ6zKyn" id="content10-3s">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Co-Diversification of an Intestinal Mycoplasma and Atlantic Salmon</strong><br></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Jacob Agerbo Rasmussen [University of Copenhagen]</strong><br><br><em>Coauthors: Pia Kiilerich, Abdullah S Madhun, Rune Waagbø, Erik-Jan R Lock, Lise Madsen, M Thomas P Gilbert, Karsten Kristiansen, Morten T Limborg</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Understanding the evolutionary relationships between a host and its intestinal resident bacteria can transform how we understand adaptive phenotypic traits. The interplay between hosts and their resident bacteria inevitably affects the intestinal environment and, thereby, the living conditions of both the host and the microbiota. Thereby this co-existence likely influences the fitness of both bacteria and host. Whether this co-existence leads to evolutionary co-diversification in animals is largely unexplored, mainly due to the complexity of the environment and microbial communities and the often low host selection. We present the gut metagenome from wild Atlantic salmon (<em>Salmo salar</em>), a new wild organism model with an intestinal microbiota of low complexity and a well-described population structure, making it well-suited for investigating co-evolution. Our data reveal a strong host selection of a core gut microbiota dominated by a single Mycoplasma species. We found a clear co-diversification between the population structure of Atlantic salmon and nucleotide variability of the intestinal Mycoplasma populations conforming to expectations from co-evolution between host and resident bacteria. Our results show that the stable microbiota of Atlantic salmon has evolved with its salmonid host populations while potentially providing adaptive traits to the salmon host populations, including defence mechanisms, biosynthesis of essential amino acids, and metabolism of B vitamins. We highlight Atlantic salmon as a novel model for studying co-evolution between vertebrate hosts and their resident bacteria.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pQFyMok" id="content10-3t">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Soil Microbiome Histories Impact Plant-Microbiome Interactions and Plant Phenotypes under Drought-Stress</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Nichole Ginnan [Pennsylvania State University]</strong><br><em><br>Coauthors: Valéria Custódio, David Gopaulchan, Natalie Ford, Gabriel Castrillo, Maggie R. Wagner</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Soil microbes can adapt in free-living or plant-associated environments. We explored how microbiome water availability and host-association history impacts bacterial taxa and downstream community-level traits. Prairie soils with low/high precipitation histories were collected. Under simulated drought, soil communities from dry sites increased <em>Tripsacum dactyloides</em> (gamagrass), a native grass and maize relative, growth, compared to wet site soil microbiomes. This indicates that our soil microbiomes are adapted to local precipitation and these legacy effects impact native plant drought tolerance. Next, we conditioned these soil microbiomes for five months in one of four treatments: a factorial combination of +/- water-stress and +/- plant host (gamagrass) – collecting meta-genomic/-transcriptomic data for the input and post-conditioning microbiomes. The output microbiota were extracted/inoculated onto maize and gamagrass seedlings to test the functional consequences of microbiome legacy and recent conditioning on native/non-native plants under water-stress. Both legacy and conditioning significantly impacted the test plants’ root microbiome composition, with legacy being a stronger predictor of community composition than the conditioning environment. Maize root microbiome composition was more strongly influenced by different treatment groups than gamagrass communities. Lastly, our preliminary data indicates that microbiome histories differentially impacted maize/gamagrass phenotypes – with specific bacterial taxa as key drivers.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pRA4Vwx" id="content10-3u">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Insects (and their Microbiome) as Food and Feed – Challenges and Solutions</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Assoc. Prof. Simon Bahrndorff [Aalborg University]</strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Insects have a high potential for converting organic waste- and by-products into food and feed for a growing human population due to symbiosis with microorganisms. These symbioses provide an untapped reservoir of functional microbiomes that potentially can be used to improve industrial insect production but are poorly studied in most insect species. The realization that insect microbiomes play a significant role in the life of insects has led to increased interest in exploiting host-microbiome interactions by the insect farming industry. Our work aims to explore the biotechnological applications of the holo-genome for improving insect production. Through a better understanding of the functional role of the gut microbiomes in two candidates for commercial insect production (the black soldier fly (<em>Hermetia illucens</em>) and house fly (<em>Musca domestica</em>)) we explore on the applied use of microbiomes in insect production. On the long term, we expect to be able to shape relevant and beneficial insect microbiomes and by doing so better utilize different waste- and by-products for a more sustainable future food system.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pS3Bb0o" id="content10-3v">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Crop Drought Resilience in the Era of Hologenomics</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Asst. Prof. Chris Barnes [Aarhus University]</strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">There is growing awareness of the importance that microbes play in regulating plant phenotypes, with specific microbes associated with increased drought resilience. Historically, the potential of crops to withstand environmental stresses, such as droughts, has focused on the host genome alone. More recently, there has attempts to manipulate plant microbiomes to enhance crop resilience, either by promoting diverse soil microbiomes or the addition of probiotics. With a growing acceptance that genome variation effects plant-associated microbiomes, many have hypothesised that crops can be bred which cultivate root microbiomes that provide more assistance in resisting drought stresses, but this has yet to be implemented. Currently, our European Commission funded project BarleyMicroBreed aims to make this a reality. In this project, we are performing a systematic study of 600 barley lines, correlating genome variation with field measures of the root microbiome and phenotype data (from drought prone and temperate field sites). We hope to associate genes that regulate the root microbiome, which in turn effects the host’s drought tolerance. This interaction can then be validated in the laboratory using CRISPR. Finally, these genes can be bred into high yielding lines using speed breeding, generating a new barley line with an improved root microbiome.&nbsp;</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pSyYMLd" id="content10-3w">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Analysis of Variability through the Relative Site Frequency Spectrum (rSFS): Applications to Host-Microbiota Interactions</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr. Sebastian E. Ramos-Osins [Centre for Research in Agricultural Genomics]</strong><br><br><em>Coauthors: Minlong Chen, Daniel Crespo-Piazuelo, Maria Ballester, Yuliaxis Ramayo-Caldas</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Recent evidence suggests that a reduction of host-genetic variability constrains microbiome diversity. These results point out to consider population genomics tools to understand the mechanism involved in host-microbiome interactions. We have developed estimators of variability per haplotype (and for genotype/individual), in relation to the relative Site Frequency Spectrum (rSFS) of the present mutations in the sampled population. The relative rSFS counts the frequency of each variant concerning the population and can be used to compute different estimates of variability, differentiation, and may also be useful detecting co-evolutionary patterns. We evaluate the performance of this methodology in discerning differential patterns among individuals, through simulation analyses of population variability: (i) considering selective effects and (ii) considering differential gradient patterns in a spatial context. Finally, (iii) we adopt the term signatures of holobiont diversity (SHD) and propose a methodological framework to explore links between rSFS and gut microbial diversity.  For this purpose, we used whole-genome sequencing data and gut microbial information from 100 Duroc pigs. We identified signals of SHD comprising genes related with relevant homeostatic processes like chromatin modification (MACROH2A2), stability of mRNAs (EIF4ENIF1), intracellular trafficking (MTMR2), and mucosal immunity (TRIM16, MMP7). Taken together, our findings support the links between patterns of host-genome diversity and the richness of porcine gut microbiota, thereby suggesting the usefulness of SHD in understanding host-microbial interactions. We anticipate that, beyond pigs, our approach can be extended to other holobiont, favouring the implementation of novel management strategies in conservation and breeding programs.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pTaw3QW" id="content10-3x">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Nematode Holobiont – Role of Microbes in Nematode Suppression and Protection</strong><br></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Olivera Topalovic [University of Copenhagen]</strong><br><br><em>Coauthors: Holger Heuer, Mette Vestergård, Stefan Geisen, Flemming Ekelund</em><div><br></div></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Nematodes are among the most abundant animals in the soil, occupying different trophic levels. Free-living nematodes (FLNs) are involved in soil nutrient cycling by feeding on microbes. Plant-parasitic nematodes (PPNs) affect plant performance by feeding on plant tissues and also by forming pathobiome complexes with microbiomes. Over the past decade, it has been increasingly recognised that nematodes specifically associate with the microbiomes that make up the nematode holobiont. These microbiomes can suppress nematodes through various forms of antagonism, but can also protect nematodes from antagonists or promote their establishment on roots. Here, we combined classical microbiological techniques and next-generation amplicon sequencing to determine the nature and composition of nematode-associated microbiomes. We show how soil microbial competition and plant diversity contribute to shaping nematode-microbe complexes and nematode performance. We also highlight the importance of PPN and FLN microbial associations for plant health. Overall, our studies contribute to a better understanding of the nematode holobiont and its role in nematode and plant performance.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7VGbIXwE" id="content10-3y">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Genome-Resolved Metagenomic Analysis of the Gut Microbiome in Reared House Fly Larvae (<em>Musca domestica</em>)</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Asmus Toftkær Muurmann [Aalborg University]<br></strong><br><em>Coauthors: Jacob Agerbo Rasmussen, Morten Tønsberg Limborg, Tom Gilbert, Simon Bahrndorff</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">During the last decades there has been an increasing focus on utilizing insects for food and feed. Especially black soldier fly and housefly larvae are interesting organisms due to their potential in converting low value biomaterial into protein. One central question is what role the microbiome plays on insect performance and how the microbiome can be utilized to improve insect health and performance. In this study larvae of the housefly (<em>Musca domestica</em>) were reared on three substrates, their growth performance measured, and their microbiome sequenced. A catalogue of metagenome assembled genomes (MAGs) was constructed with hybrid assembling using both Illumina and Oxford Nanopore reads. The MAG catalogue was used to study the effect of substrate on microbiome composition and metabolic capacity as well as correlations between larval growth performance and microbial dynamic, including taxonomy and functionality. Results clearly reveal effect on the microbial dynamics by diet mediums. Our investigations reveal MAGs, which are highly related to the intestinal environment and that these MAGs affect the growth phenotypes of the house fly larvae, including survival and feed conversion. Subsequent investigations on the metabolic capacity indicate a lack of correlation between taxonomy and functionality - a result that should cause reflection given the widespread use of amplicon-based methods in insect rearing research.&nbsp;</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3IGyBAMJ" id="content10-ax">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><br><div><strong>Long-read Metagenomics of Shanghai Pet Dogs Enables the Retrieval of Hundreds of Bacterial Species as High-Quality MAGs</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Anna Cusco [Fudan University]</strong><br><br><em>Coauthors: Yiqian Duan, Fernando Gil, Shaojun Pan, Xing-Ming Zhao, Luis Pedro Coelho</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Pet owners consider their animals family members and care about their health. However, most studies on dog microbiomes focus on dogs from colonies rather than pets in households. Using long-read metagenomics, we characterized a cohort of 50 pet dogs living in Shanghai. We collected a fecal sample and the dog-associated metadata (80+ questions). We retrieved 1,921 high-quality and 756 medium-quality metagenome-assembled genomes (MAGs) from 320 bacterial species. Long-read metagenomics recovered genomes for most microbiome members (on average, 97% and 86% of the long- and short-reads, respectively, mapped back to the species-level MAG catalog).<br>The most prevalent species were <em>Blautia hansenii, Faecalimonas umbilicata, Ruminococcus_B gnavus, Clostridium_Q sp000435655 </em>and <em>Enterocloster sp001517625</em>. Across our cohort, the Average Nucleotide Identity within each species varies widely. For example, <em>Blautia argi</em> is highly similar across the different dogs of our cohort, while <em>Blautia hansenii </em>is not. Further comparison with other cohorts will confirm whether these highly similar strains are dog-specific and how they differ from other hosts. Finally, we demonstrate that dogs living together share more bacterial strains than those living in different households. We are working to relate the metadata (e.g., size, age, or sex) with specific microbiome features.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3IHsiM4D" id="content10-ay">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Microbiome Dynamics Across the Lifespan of Australian Angus Cattle</strong><br></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Thibault Legrand [CSIRO]</strong><br><br><em>Coauthors: Pâmela Alexandre, Annaleise Wilson, Ryan Farr, Antonio Reverter, Stuart Denman</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Understanding the intricate nexus between cattle welfare and microbiome dynamics holds profound implications for enhancing animal’s health, productivity, and animal welfare. In cattle production, animals undergo a series of production boundaries inducing stress and potentially increasing their susceptibility to infectious diseases. In this regard, immune competence (IC), corresponding to a combined measure of cell-mediated (cell-IR) and antibody-mediated (Ab-IR) immune responses, is a useful disease resistance trait that can be used in conjunction to production traits to select for resilient and fast-growing cattle. In this study, we characterised the microbiome dynamics of a cohort of 67 steers with known genotype and IC across multiple timepoints including marking, weaning, pre-feedlot and post-feedlot. Metagenomic libraries were prepared using the Nextera DNA flex kit and sequenced on the NovaSeq Illumina platform. In total, we recovered 87, 348 and 1532 medium to high-quality MAGs from nasal mucus, saliva and feces samples respectively. Microbial community composition and function differed between sample types and timepoints revealing important microbial dynamics across the lifespan of cattle. By linking microbial functionality and traits such as IC and growth, we hope to better understand the interplay between the cattle and their associated microbes to further improve immune resilience in beef cattle.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3JtppuBH" id="content10-b0">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Panel Discussion:</strong></div><div><strong><br></strong></div><div><strong>Multi’omics and the Future of Sustainable Agrisciences</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Assoc. Prof. Morten Limborg [University of Copenhagen] (Moderator)</strong><div><strong>Prof. Lone Gram [Technical University of Denmark]</strong></div><div><strong>Dr Stinus Lindgreen [Radikale Venstre]</strong></div><div><strong>Prof. Claus Felby [Novonordisk Foundation]</strong></div></h6>
            </div>
            <div class="col-md-12 col-lg-5">
                <p class="mbr-text mbr-fonts-style display-7"></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content12 cid-th8p99sExk" id="content12-4g">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-md-12 col-lg-10">
                <div class="mbr-section-btn align-center"><a class="btn btn-primary display-4" href="AHC_Abstracts.html#top"><span class="mobi-mbri mobi-mbri-arrow-up mbr-iconfont mbr-iconfont-btn"></span>Back to top</a></div>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content4 cid-th808JO3Fc" id="content4-40">
    
    <div class="mbr-overlay" style="opacity: 0.7; background-color: rgb(233, 209, 210);">
    </div>
    <div class="container">
        <div class="row justify-content-center">
            <div class="title col-md-12 col-lg-12">
                <h3 class="mbr-section-title mbr-fonts-style align-center mb-4 display-1"><strong>Session 3: Biodiversity and Conservation</strong></h3>
                
                
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7VI8IXnv" id="content10-3z">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Eco-Evo-Devo Drivers of Microbial Assemblage Dynamics in Tropical Marine Holobionts</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><div><strong><span style="font-size: 1rem;"><b>Prof. Monica Medina [Pennsylvania State University]</b></span><br></strong></div><div><br></div></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Cnidarian photosynthetic holobiont members – which include the host, at least one unicellular algal symbiont and a poorly characterized consortium of other diverse microbial taxa – have different generation times and effective population sizes. These differences can have pronounced evolutionary and ecological consequences for holobiont members, especially under a rapidly changing climate. Hosts (e.g. corals, jellyfish) tend to have slow generation times relative to their microbes, on the order of decades. Algal symbionts with different physiological adaptations to light and temperature are known to shift in abundance within a host over months to years. Host associated prokaryotic microbial assemblages not only have very fast generation times, but their community composition can shift rapidly under changing environmental conditions. We have performed experiments on different coral and jellyfish holobionts across  the life cycle and physiological states. These studies have uncovered diverse responses linked to developmental and physiological phenotypes as well microbiome-associated responses helping define holobiont niche boundaries. We have only scratched the surface of complex holobiont dynamics, yet these studies highlight the need for additional data and a comprehensive theoretical framework that integrates ecological and evolutionary time scales.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th80KQ4n0U" id="content10-41">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Keep One Another Close: The Importance of Wildlife-Microbe Symbioses in a Changing World</strong><br></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Assoc. Prof. Kevin Kohl [University of Pittsburgh]</strong><br><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Anthropogenic changes to our planet have imposed numerous challenges to wildlife and their associated microbes. With increasing recognition of how microbes impact animal health, conservationists may need to consider wide-ranging factors that may disrupt or preserve these host-microbe interactions. That is, how do we conserve “holobionts”? Here, we will begin to consider this question in two systems. First, with relevance to climate change, we have demonstrated that microbes aid larval amphibians in tolerance to heat stress and heat waves. We are now working to understand landscape variation in these associations, and how human land use (pollution, agriculture) may disrupt important partnerships.  Second, given increasing needs for captive rearing programs to preserve species diversity, we are working to understand the microbial ecology of captive care and the built environment. Specifically, through collaboration with the International Crane Foundation, we are working to improve artificial rearing practices of endangered Whooping Crane chicks. We have shown that chicks raised by humans in costume have significantly altered microbiomes from those raised by adult birds. We have conducted fecal microbiome transplants (FMTs) with the goal of rescuing the microbiome and ameliorating health defects which are also more common in costume-reared chicks. Together, these studies consider need to understand and conserve host-microbe interactions in a changing world.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th80LnDIzZ" id="content10-42">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Gut Microbiome Characteristics are Associated with Conservation Practices and Host Genetic Diversity in a Wild Vertebrate Species</strong><br></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Sarah F. Worsley [University of East Anglia]</strong><br><em><br>Coauthors:&nbsp;Charli S. Davies, Zoe Crighton, Maria‑Elena Mannarelli, Terry Burke, Jan Komdeur, Hannah L. Dugdale, David S. Richardson</em><br><br><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The vertebrate gut microbiome (GM) makes fundamental contributions to host processes. However, the GM often varies extensively within and among populations of the same host species. Determining the drivers of this variation is essential for understanding the ecology and evolution of host-microbe interactions and how the GM could be manipulated to improve host health and, in conservation scenarios, population persistence. Here, we use a candidate gene approach to evaluate the relationship between host genetic variation and GM structure within a natural population of the once critically endangered Seychelles warbler (<em>Acrocephalus sechellensis</em>). We also compare GM variation within the original remnant population on Cousin Island to other populations of the species established through translocations. On Cousin Island, host immunogenetic variation was found to be significantly associated with differences in GM diversity and composition, whilst genome-wide heterozygosity (a measure of inbreeding) was associated with differences in fungal GM structure. Effect sizes were small but potentially important, as evidenced by a significant association between GM composition and host survival. GM structure also differed amongst populations; individuals on Cousin Island harboured greater GM diversity and demonstrated greater inter-individual GM variability compared to translocated populations. Future work will use whole genome sequence data to assess how host bottlenecks and inbreeding (runs of homozygosity) influence GM structure across populations. This will shed further light on how conservation practices and population declines may impact host-microbe interactions.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th80M4KeLA" id="content10-43">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Pleistocene Reindeer Dental Calculus as a Source for Ancient Oral, Rumen and Dietary Flora</strong><br></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Jaelle Brealey [Norwegian Institute for Nature Research]</strong><br><br><em>Coauthors: Fabian L. Kellner, Vanessa C. Bieker, Vebjørn Veiberg, Mikkel Winther Pedersen, Katerina Guschanski, Michael D. Martin<br></em><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Rumen microbes have essential roles in digestion in ruminants like reindeer (<em>Rangifer</em> <em>tarandus</em>). However, soft tissues like the rumen are rarely preserved in the archaeological record, posing challenges for studying host–microbiome evolution over long timescales. Recently it has been found that rumen microbes can be incorporated into the microbial biofilm of dental plaque. Plaque calcifies into calculus, which preserves DNA from oral and regurgitated rumen microbes, as well as dietary components and host DNA. We investigated the potential of dental calculus for studying the long-term evolution of the reindeer holobiont, using metagenomics to characterise microbes, plants and lichens preserved in calculus from Pleistocene reindeer from extinct French populations (15000–25000 years BP) and modern reindeer from two extant wild Norwegian populations. We found that microbiome and diet compositions differed temporally and spatially. However, five rumen microbes and several known dietary plants of extant reindeer were detected in both modern and ancient samples, whereas important reindeer lichens were only detected in modern populations. Our results demonstrate the value of dental calculus for studying holobiont evolution, including oral and rumen microbiomes of extinct populations. Such knowledge will improve our understanding of the ecology and evolution of vulnerable ruminant holobionts, aiding conservation strategies.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th80MGWMrn" id="content10-44">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Temporal Hologenomics: A New Approach in Biodiversity Research, Conservation and Evolutionary Biology</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Assoc. Prof. Katerina Guschanski [University of Edinburgh]</strong><em><br></em><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Change is an underlying principle of all life. In the last decades, humans are increasingly the predominant drivers of environment and ecological change, leading to loss of biodiversity, population decline, and contamination of the environment with human-made products. While the impacts of these changes are broadly recognised across many disciplines, how they affect host-microbiome interactions has received little attention. One of the main reasons for this “neglect” is the lack of baseline host-associated microbial samples from the past, which would allow us to quantify, characterise, and understand the consequences of human-mediated environmental change. Dental calculus, the calcified form of the oral biofilm, is a microbial fossil that preserves over millennia and offers such a window into the past. Dental calculus deposits appear to be widely present across the mammalian tree of life, allowing for ecological and evolutionary studies in diverse species. The composition and function of dental calculus microbiomes reflect host ecology, diet, and phylogeny and appear to be structured by the environmental niches that the hosts present, without strong evidence for phylosymbiosis. Understanding these underlying principles of dental calculus microbiome evolution allows the explorations of more pressing questions, aiming to understand the effects of human-driven population declines and environmental contamination on host-microbiome interactions and their consequences for the hosts. I will provide examples for how temporal hologenomics can generate first insights into these processes.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3KtWYC2W" id="content10-b1">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>The Hologenomics of Invasion Biology</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Senior Scientist Cornelia Jaspers [Technical University of Denmark]</strong><em><br></em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Invasive species represent a major threat to biodiversity and ecosystem functioning. Human mediated transport is accelerating the rate of species translocations, leading to a homogenization of the worlds' biota. Surprisingly, only a small fraction of the daily transported organisms become invasive. This leaves the question: What makes a species successful to invade novel habitats? I will present general principles of invasion biology, genomic approaches allowing to unravel invasion dynamics and the potential contribution of the microbiome to host fitness in an invasion context. To do so, one of the most prominent marine invasive species is used as model organism. Whole genome resequencing analyses of native and invasive populations are combined with experimental evolution studies and microbiome analyses. Specific focus is devoted to changes in the host performance along environmental stress gradients and subsequent modifications of the core microbial community across populations. Swift selection for increased host fitness along with selection for indicator microbes was observed. These correlative results hint to the potential importance of both the host genotype and its microbiome, contributing to fitness and invasion success. A hologenomic approach to invasion biology might help to unravel so far largely ignored aspects and processes of species´ fitness under global change.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3KuD1W45" id="content10-b2">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>ASG Project: First Insights about Microbial-Rich Sponge Metagenomes</strong><br></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Camilla Santos [Wellcome Sanger Institute]</strong><br><br><em>Coauthors: Genome Reference Informatics Team<br></em><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The Aquatic Symbiosis Genomics (ASG) project aims to provide comprehensive genomic data for ecologists and conservationists to understand how species evolve and live together, in addition to how the symbiosis process may influence aquatic environment dynamics. Although ASG includes several aquatic species, sponges (Porifera) have the richest symbiotic microbiota, due to their filter-feeding nature, compounding the ASG sponge-microbe symbioses hub. Sponges and microbiota genomes are assembled separately and host genomes are curated to become high-quality chromosome level assemblies, while cobionts are submitted as metagenomes assemblies (MAGs), which are largely complete and representative of a single species. Due to the great amount of cobionts living in sponges, the host genome assembly and curation may be challenging, with sometimes more than 50% of assembly length being composed of cobionts. Currently, ASG has 24 sponge genomes submitted, of which 22 have MAGs, totalizing 877 sponge metagenomes submitted and many more coming. We have observed a wide diversity of prokaryote species in the MAGs, where  the top 5 bacteria groups are Dehalococcoidia, Alphaproteobacteria, Gammaproteobacteria, Pseudomonadales and Poribacteria. Many of them might work as aquatic bioindicators and are also known for bringing some benefits to the host, such as production and absorption of metabolites and defence against pathogens and predators. Making available and public sponge and cobiont genomes at large scale is one of ASG targets, providing data for a better understanding of microbiome complexities and functionalities and their potential application in healthy, conservation and technological areas.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3KvkrLPm" id="content10-b3">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>The Role of Microbial Symbiosis in Insular Adaptation: A Hologenomic Approach</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Laura Baldo [University of Barcelona]</strong><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Insular populations are a great natural system to study evolutionary processes, as well as major targets of conservation studies due to their high rate of endemism and current challenge by habitat destruction. Yet, the study of microbial symbiosis in host insular adaptation and population resilience is still missing. Here we will present the Balearic wall lizard, <em>Podarcis lilfordi</em>, as a holobiont model to address the role of gut microbial symbionts in animal fitness landscape and particularly in their ability to offer host metabolic adaptation in a resource-constrained environment. We count with a recently sequenced genome, long-term individual-based data, genetic data based on Single Nucleotide Polymorphisms (SNPs), demographic and ecological data (diet) and the description of the gut microbiota by 16S partial sequence for multiple populations.  We will present preliminary data based on 16S full-length sequence by Oxford Nanopore Technology (ONT) on 15 populations (280 individuals) from the archipelagos of Menorca, Mallorca, and Cabrera, exploring intra and interpopulation differences, host-specificity, seasonal plasticity and predicted functional aspects of the gut microbiota. Our results provide a critical example of data integration for a natural system and aim at a comprehensive understanding of the microbial symbiont contribution to vertebrate insular adaptation and persistence.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3KvY0d5Y" id="content10-b4">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>A Comprehensive Investigation of Woolly Mammoth Remains Associated Microbes</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Benjamin Guinet [Swedish Museum of Natural History]</strong><br><br><em>Coauthors: Nikolay Oskolkov, Love Dalén, Tom van der Valk</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Throughout history, the exploration of ancient DNA (aDNA) has primarily centered on the evolution and demographics of humans and animals, emphasizing the analysis of eukaryotic aDNA. However, the discovery of microbial sequence data has revolutionized this field by uncovering the significance of host-associated microbial aDNA found within eukaryotic remains. Once considered a mere sequencing by-product, this microbial aDNA has emerged as a crucial source of insights into ancient pandemics, lifestyle patterns, and population movements. Despite the broad potential of obtaining evolutionary microbial insights from ancient DNA remains, the vast majority of studies to date have remained focused on humans. In this study we identified the microbial DNA found within 495 woolly mammoth remains, comprising tooth, tusk and skins samples, collected from a broad geographic range and timescale spanning the holocene (~4300 years old) up to samples over a million years in age. This extensive dataset enables us to assess and formulate hypotheses regarding numerous potential interactions between these ancient microorganisms and their hosts. Among the identified microorganisms, several related bacteria are known to interact with contemporary animals in either commensal or pathogenic ways. This finding paves the way for a deeper understanding of the diseases that might have affected mammoths and their impact on mammoth populations evolution through time.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content12 cid-th8pajXb16" id="content12-4h">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-md-12 col-lg-10">
                <div class="mbr-section-btn align-center"><a class="btn btn-primary display-4" href="AHC_Abstracts.html#top"><span class="mobi-mbri mobi-mbri-arrow-up mbr-iconfont mbr-iconfont-btn"></span>Back to top</a></div>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content4 cid-th84ulepkG" id="content4-45">
    
    <div class="mbr-overlay" style="opacity: 0.7; background-color: rgb(233, 209, 210);">
    </div>
    <div class="container">
        <div class="row justify-content-center">
            <div class="title col-md-12 col-lg-12">
                <h3 class="mbr-section-title mbr-fonts-style align-center mb-4 display-1"><strong>Session 4: Future Technical Trends</strong></h3>
                
                
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th84zzc6ZV" id="content10-46">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Population Theory for the Hologenome and the Assembly of Holobionts</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Joan Roughgarden [University of Hawaii]</strong><br><em>&nbsp;<br></em><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The hologenome is the union of genes carried by microbes with the genes carried by chromosomes in the host nucleus. The power of holobiont selection acting on the hologenome to explain microbiome-host integration is limited by the absence of a Hardy-Weinberg law for microbial genes in contrast for nuclear genes. Hence, directional selection on microbial genes is not as effective as that on nuclear genes. Instead, microbiome-host integration appears best explained by a process of adaptive assembly called Host-Orchestrated Species Sorting (HOSS). Moreover, a phage becomes lysogenic and integrates into the genome of a microbe if the multiplication rate of the microbe exceeds the expected viron production by the phage if it were lytic. A tripartite holobiont results if a phage integrates into a microbe that in turn enters the host via host-orchestrated species sorting.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th84ABhSvj" id="content10-47">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong><em>TBA</em></strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Ana Pombo [Max Delbrück Center for Molecular Medicine]</strong><br><br><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7"><em>Abstract Coming Soon</em></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th84Bli8PJ" id="content10-48">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Intertwining Plasmids, Microbial Interactions and Adaptations to Gut Environments</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Itzik Mizrahi [Ben Gurion University]</strong><br><em><br></em><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Gut environments, densely populated with microorganisms, serve as fertile grounds for horizontal gene transfer and microbial genome plasticity, with plasmids emerging as potent agents in this evolutionary process. This prompts inquiries into the driving forces behind plasmid dispersal across populations, the intricate factors shaping genetic material flow among them, and their impact on microbial interactions and host adaptation. In my presentation, I will talk about findings from our recent studies that probe these questions, offering deeper insights to these aspects of microbial ecology and evolution.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th84BUdEqL" id="content10-49">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Unveiling Animal-Microbiota Interactions through the Study of Metabolic Capacities of Microbial Genomes and Metabolite Exchange Capabilities of Microbiomes</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Assoc. Prof. Antton Alberdi [University of Copenhagen]</strong><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">While the significance of microorganisms in supporting the normal biological functioning of animals is widely acknowledged, the mechanisms underlying these interactions remain elusive.  The reconstruction of microbial genomes from complex DNA mixtures such as faeces using genome-resolved metagenomics, brings us the opportunity to characterise their functional capabilities at an unprecedented resolution. Harnessing this technology, we have recently developed two R packages that quantify metabolic features of microbial genomes and communities using two complementary approaches. distillR facilitates the distillation of gene annotations into genome-inferred functional traits, while metaDEEP quantifies metabolite exchange capacities of communities from genome-scale metabolic networks. I will explain the basis of these two complementary software and showcase them in diverse study systems to address a number of biological questions partaining animal-microbiota interactions.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3LyKktev" id="content10-b5">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Exploring Evolutionary Host-Microbiota Interactions across Vertebrates Using Gut-On-a-Chip</strong><br></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Asst. Prof. Ostaizka Aizpurua [University of Copenhagen]</strong><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Gut microorganisms influence many biological functions of their vertebrate hosts, a fact that led many researchers to posit that host-microbiota interactions have likely shaped the evolutionary trajectories of vertebrates. Although it is widely acknowledged that gut microbiomes are shaped by host phylogeny and dietary preferences, there is a notable scarcity of direct evidence to substantiate that these interactions might have shaped animal evolutionary trajectories. This knowledge gap is primarily attributed to technical challenges associated with conducting inter-specific comparative analyses of how microbes are detected and which reactions these detections trigger across vertebrates. In that regard, I argue that cutting-edge Gut-On-a-Chip (GOC) technology, mainly used in human health and pharmacological research, can enable us to tackle previously unattainable questions. GOC is a next-generation in vitro model that mimics the essential functionalities of their<em> in vivo</em> counterparts. Considering its potential, we are generating multi-species GOC models to perform the first comparative evolutionary studies to measure whether and how the ability to recognise different microbial stimuli and the capacity to use microbe-produced metabolites varies across vertebrates, thereby increasing our understanding of species-specific host-microbiota interactions.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3LzQtKBw" id="content10-b7">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Epigenetic Control of the Gut Microbiome</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Assoc. Prof. Morten Limborg [University of Copenhagen]</strong><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Emerging evidence across all areas of life has revealed the evolutionary importance of the intimate biological interactions between animals and their associated microbiota. Both the host genotype and the host microbiota have been shown to influence host phenotypes, such as growth and disease resilience. However, only little is known about the molecular mechanisms by which this host-microbiota crosstalk happens. I will present how we combine the zebrafish system with long read sequencing and CRISPR based manipulations to study the causative mechanisms by which host species interact with their microbiota communities. We test effects of both specific host genes as well as host epigenomic traits in shaping the intestinal microbiota and its metabolic functions. Using CRISPR we show that regulated activity of e.g. the tyrosinase (tyr) gene affects the gut microbiota composition. We also show that zebrafish hosts can respond to external stressors (e.g. famine) by eliciting an epigenetic response mechanism that extends into a so-called epigenome – microbiome axis. With this presentation I aim to demonstrate the value of implementing Nanopore sequencing and CRISPR into the hologenomics concept to better understand the evolutionary importance of animal-microbiota interactions.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3LAOl5Uv" id="content10-b9">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Deep Viral Metagenomics Reveals Dynamic Changes in Mobile Element Interaction Networks Following Intestinal Microbiota Transplantation</strong><br></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Benjamin Auch [Phase Genomics]</strong><br><br><em>Coauthors: Samuel Bryson, Benjamin Auch, Jonas Grove, Bradley Nelson, Ivan Liachko, Alexander Khoruts</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Intestinal microbiota transplantation (IMT) has proven effective for recurrent <em>Clostridioides difficile</em> infection (rCDI). However, the analysis of microbiome response has been hindered by limitations in 16S and metagenomic shotgun sequencing, particularly in associating mobile elements (plasmids and phages) with their microbial hosts. Studies using bacteria-free filtrates from fecal suspensions have shown comparable efficacy to whole fecal material in rCDI treatment, suggesting the involvement of phages and plasmids in gut remodeling. Proximity ligation sequencing (Hi-C) is an approach that crosslinks genomic material within cells, capturing physical interactions between the host microbial genome and mobile elements. In addition to recovering highly complete microbial genomes, proximity ligation approaches allow for the recovery of the phage-host and plasmid-host interactome. We employed this approach with a cohort of 48 patients receiving IMT from a single donor, sampling over several weeks following IMT delivered using oral capsules. Our analysis reveals unique, patient-specific patterns of engraftment, a significant increase in prokaryotic genome diversity, temporal changes in the biosynthetic capacity of the gut, and a re-configuration reminiscent of the donor. Phage-bacteria interactions also exhibit patient-specific patterns. The observed increase in phage diversity post-IMT aligns with that seen in the microbial populations, and is linked to multiple sources, including a donor-derived influx and baseline-derived persistent phages. In some cases, engrafting phages locate new hosts, a poorly understood phenomenon standing in contrast to a canonical “one phage, one host” model. Finally, we show that a failure in initial engraftment of phages may be predictive of IMT failure.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3LBgINFJ" id="content10-ba">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Holo-omic Workflow Utilizing Network Analysis Reveals Compositional Bistability Between Rumen Microbiomes</strong><br></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Carl Mathias Kobel [Norwegian University of Life Sciences]</strong><br><br><em>Coauthors: Wanxin Lai, Velma T. E. Aho, Arturo V. P. de Leon, Ove Øyås, Laura Nicoll, Andy Leu, Gene W. Tyson, Simon J. McIlroy, Ianina Altshuler, Torgeir R. Hvidsten, Rainer Roehe, Phil B. Pope</em><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The holobiont concept emphasizes analyzing the host and its microbiome as a whole. In this context, “holo-omics” presents quantitative methods that can be used to investigate host-microbiome data and facilitate predictions on biological interactions. Here, we deeply sampled 24 beef cattle (2 breeds, 30% rel. difference in CH4) and their microbiomes across rumen contents, rumen epithelium, and liver tissues. From all samples several molecular layers were generated in order to create a high-resolution dataset. To reconstruct metagenome-assembled genomes (MAGs) that could detect strain-level variation we applied long-read Oxford Nanopore Technology (ONT) r10.4 flowcells. Multi-omic analyses established that there appears to be a ubiquitous binary type of strain composition that was defined across two distinct groups of animals and is enriched with intermediate fermentative bacteria and methanogens. Interestingly, our investigations to date have shown that neither strain-defined animal group is associated to animal breed nor any measured key performance indices. We’re developing a fully parametrized holo-omic workflow based on weighted gene co-expression network analysis (WGCNA). This workflow enables optimizing parameters for clustering of correlated genes. These clusters can then be linked with biological processes and in case of meta-omics, we can investigate the roles of specific microbial species as well. We are using this workflow to investigate the host response of the “microbiome split” in the rumen wall and we are also looking into whether this signal can be picked up further downstream the nutrient assimilation pathway in the liver.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3LAj9jqy" id="content10-b8">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><br><div><strong>Advancements in Applying Zebrafish as a Model Organism in Hologenomics</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Eiríkur Andri Thormar [University of Copenhagen]</strong><br><br><em>Coauthors: Heidi Mathiessen, Søren Blikdal Hansen, Michael L. Pepke, Louise von Gersdorff Jørgensen, Morten T. Limborg</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The zebrafish serves as an attractive model organism in the field of hologenomics for several reasons. Firstly, it is a vertebrate animal model that is easy to work with, has a short generation time, and is relatively easy to genetically manipulate. Secondly, the popularity of the model across various fields of science has resulted in a plethora of available scientific resources. Using the CRISPR/Cas genetic engineering technique we investigated the potential of using zebrafish in a hologenomic context, specifically to address the potential impact of host genes on the composition of the zebrafish microbiota. By knocking out the genes tyr, irf8, and cxcl8  and comparing the microbiota of knockout zebrafish to non-knockout counterparts, we demonstrate the utility of the model, while uncovering potential pitfalls and experimental considerations. Additionally, we share results from our preliminary investigations that identify potential mechanisms by which the host influences its microbiota composition. Through the use of high throughput sequencing techniques, we highlight aspects surrounding the observed differences in the knockouts and non-knockout counterparts, the importance of sampling technique, appropriate controls, and the choice of candidate genes.&nbsp;</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3LzhQlPX" id="content10-b6">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Spatially Resolved Multi-omic Landscape of the Animal Gut Microbiome</strong><br></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Bryan Wang [University of Copenhagen]</strong><br><br><em>Coauthors: Zoé Horisberger, Carlotta Pietroni, Lorrie Maccario, Carlos Llamas, Ida Thøfner, Soile Turunen, Andi Erega, Annelies Geirnaert, Olli Kärkkäinen, Anders Miki Bojsen, Antton Alberdi, Søren Johannes Sørensen, Urvish Trivedi</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Spatial organization is a pervasive feature of ecosystems, and the animal gut is no exception. Hidden within animals are bacteria associated with the intestinal tract, organized in a complex structure. While much has been done to explore the large-scale diversity of the gut microbiota, the fine-scale spatial organization of this complex ecosystem remains largely unexplored. Current approaches used to study the microbiota from fecal material or intestinal contents require homogenizing the biological material. In consequence, these methods are unable to reveal the gut microbial biogeography and fine-scale spatial structure of microbial communities. In the emerging field of spatial-omics, 3D’omics aims to generate spatially resolved multi-omic information by embedding and processing samples in ways that preserve their 3D relationship. As a starting point, we use a monogastric terrestrial poultry model, to carry out multi-omic analysis of decreasing sample sizes to find the lower limit of detection of each method. We also employ fluorescence in situ hybridization (FISH) and confocal microscopy to visualize bacterial taxa at a single-cell level to complement the multi-omic data generated. Synthesizing the information generated here will allow us to move towards spatially referenced metagenomics and metatranscriptomics to understand the 3D’omic landscape in the chicken gut further.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3LBRQwwo" id="content10-bb">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Closing Keynote: </strong></div><br><div><strong>Winds of Change - Breathing a New Disciplinary Matrix into Biology</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Seth Bordenstein [Director Penn State University Microbiome Center]</strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Over the last two decades, symbiotic associations between hosts and microbes have precipitated a fresh perspective of the hierarchy of biological systems. Once an idea on the fringe of biology, the holobiont framework is now recognized as a new unifying theme in biology and in the pyramid of life's organization. Consequently, theory, experiments, organizations, and lexicons are modernizing to embrace a disciplinary matrix that prescribes prominence to the increased explanatory power and utility of realizing host life as a holobiont.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content4 cid-ue3NJDXCRw" id="content4-bc">
    
    <div class="mbr-overlay" style="opacity: 0.7; background-color: rgb(233, 209, 210);">
    </div>
    <div class="container">
        <div class="row justify-content-center">
            <div class="title col-md-12 col-lg-12">
                <h3 class="mbr-section-title mbr-fonts-style align-center mb-4 display-1"><strong>Posters</strong></h3>
                
                
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3NQ91mOJ" id="content10-be">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Gut-On-A-Chip<em>&nbsp;In Vitro</em>&nbsp;Models to Study Host-Probiotic Interactions [1]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Kees Blijleven [University of Copenhagen]</strong><br><br><em>Coauthors: Michael Gerard Miskelly, Carlotta Pietroni, Urvish Trivedi, Bryan Wang, Antton Alberdi, Ostaizka Aizpurua</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Consumption of probiotics is increasing worldwide to promote gut health. Due to the limitations to experiment with humans, much of our understanding of host-probiotic interactions derives from animal studies. Although these studies have led to significant advances, the species-specific nature of host-probiotic interactions has posed challenges to translating the acquired knowledge from model organisms into practical applications. Therefore, the efficacy and mechanism of action of most probiotics remain poorly understood. Gut-on-a-chip <em>in vitro</em> models developed to simulate the structure, function, and microenvironment of the human gut emerge as promising alternatives to dig into human-probiotic interactions. In this study, we analysed the effect of a commensal <em>E. coli </em>strain on human cells via their surface compounds using gut-on-a-chip technology. Gut epithelial cells were co-cultured with gut endothelial cells for 7 days at 30 μl/h flow rate and 10% stretching at 0.15 Hz to mimic the gut lining. Subsequently, chips were inoculated with or without heat-killed <em>E. coli </em>and kept in culture for three days. RNA-sequencing of the tissue revealed that the structural components of bacteria produced differences in the expression of human genes.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue4xUFUJ7j" id="content10-bo">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Unraveling Host Regulation of Gut Microbiota through the Epigenome-Microbiome Axis [2]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Michael L. Pepke [University of Copenhagen]</strong><br><br><em>Coauthor: Søren B. Hansen, Morten T. Limborg</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The influence of the gut microbiome on host physiology, including metabolism, immune function and behavior, is being understood in increasing detail, but less is known about how the host is actively involved in manipulating its microbiome. Recent examples of dynamic interactions between a host’s epigenotype and the composition or activity of its associated gut microbiota suggest the opportunity of the host to shape its microbiome through epigenetic feedback mechanisms, non-coding RNAs and changes in gene expression in response to, for example, environmental or microbial cues. We suggest that a bidirectional “epigenome-microbiome axis” is emerging, which embeds environmentally induced variation, and which may influence the adaptive evolution of host-microbe interactions. The epigenome-microbiome axis can be understood and investigated within a holo-omic framework with potential applications in the applied health and food sciences. Furthermore, host epigenetic control could facilitate maintenance of intestinal homeostasis, selection for microbial traits influencing host growth, reproduction or disease resistance (fitness), and ultimately the evolution of symbiosis. Here, we present an experimental framework for how host methylomes can actively be modified using a CRISPR/Cas system fused to various epigenetic effectors to potentially shape microbiome-induced phenotypes in a zebrafish model.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue9bCX4cpI" id="content10-c1">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>The Microbiome, Metabolome, and Antimicrobial Potential of <em>Mondia whitei</em>, a Herbal Medicinal Plant in East Africa [3]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Expedito Olimi [University of Southampton]</strong><br><br><em>Coauthors: Regina Wuggenig, Carolina Lobato, Samuel Bickel, Peter Kusstatscher, Wisnu Adi Wicaksono, Angelika Schaefer, Danny Coyne, John Adriko, Tomislav Cernava, and Gabriele Berg</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">African communities rely on <em>Mondia whitei</em>, a medicinal herb in the treatment of various illnesses, including potential remediation of male sexual dysfunction. The microbiome as a crucial component of the plant holobiont contributes towards host functions like metabolite production. However, fewer studies have attempted to link the plant microbiome with the metabolome which is the focus of this study. Root and rhizosphere were obtained from five geographical locations in Uganda, processed, followed by community profiling by high-throughput sequencing of fungal (ITS) and bacterial (16S rRNA) genes; quantitative PCR for estimating microbial abundance, and metabolome assessment by gas/liquid chromatography-mass spectrometry. Sample location more than plant compartment had a greater impact on microbiome composition. The samples generally contained a high abundance of microbiota (i.e., bacterial=1011 and fungal=1010), with diversity ranging between 11522 and 2888 ASVs for bacterial and fungal communities, respectively. Plant roots were rich in metabolites like cyclotetrasiloxane (octamethyl-), monobutyl phthalate, 2-hydroxy-4-methoxybenzaldehyde, xanthin, and benzaldehyde (4-methoxy-), which exhibited strong correlation with the microbiome and could be used to delineate between sample locations. Taken together, the current findings provided insights into the plant microbiome and investigated the relationship between plant metabolites and the microbiome of endangered medicinal plants like <em>M. whitei</em>. Along with its ramifications, the work serves as a primer for the sustainable future utilization of<em> M. whitei</em>'s microbial and metabolite resources.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue4xSHL6Lu" id="content10-bj">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Consequences of Wheat Dwarfing on the Rhizosphere Microbiome [4]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Monique Smith [Swedens University of Agricultural Sciences]</strong><br><br><em>Coauthors: Ian Clark, Vanessa N. Kavamura, George Lund, David Hughes, Tim H. Mauchline</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The development of dwarf wheat cultivars has revolutionised cropping systems and improved yield outcomes. However, little is known about how this modification has impacted the ability of semi-dwarf wheat varieties to interact with soil microbial communities compared with their tall ancestors. We aimed to investigate the impact of wheat dwarfing on the structure and predicted function of bacterial communities using 16S rRNA gene amplicon sequencing and shotgun metagenomics from the rhizosphere of tall and semi-dwarf wheat varieties. Amplicon sequencing revealed that wheat dwarfing influenced the rhizosphere bacterial communities. Tall varieties harboured lower diversity and richness in their rhizosphere compared with semi-dwarf varieties, but they were enriched with OTUs assigned to genera normally associated with plant growth promotion. Whereas semi-dwarf varieties were enriched in taxa more associated with bulk soil rather than the wheat rhizosphere. Metagenomic gene sets were also distinct in the rhizosphere bacterial communities from tall and semi-dwarf wheat varieties. The microbiomes of tall wheat varieties are enriched in ABC transporters, type III secretion systems as well as genes involved in signalling such as quorum sensing. Overall, our results show that wheat dwarfing has resulted in plants less able to select and sustain a complex rhizosphere.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue4xRzSGI8" id="content10-bg">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Effects of Plant-Based Proteins and Handling Stress on Gut Microbiota and Metatranscriptome in Rainbow Trout [5]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Marvin Suhr [Christian-Albrechts-University Kiel]</strong><br><br><em>Coauthors: Finn-Thorbjörn Fichtner-Grabowski, Henrike Seibel, Corinna Bang, Andre Franke, Carsten Schulz, Stéphane Céline Hornburg</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">To promote sustainable and economically viable aquaculture, plant-based protein alternatives to conventional fishmeal-based diets are being developed. In addition, with increasing concerns about animal welfare standards, stress-reduced rearing environments need to be established in modern aquaculture management processes. By combining 16S rRNA gene sequencing, RNA sequencing of bacterial transcripts and analysis of molecular stress markers, the MetaBac project aims to understand the combined effects of plant proteins and handling stress on the gut microbiome of rainbow trout and the contribution of the gut microbiota to stress physiology and nutrient utilisation. We conducted a feeding trial in which trout were fed a plant-based diet while being exposed to a daily handling stressor. At the end of the trial, we found that specific molecular stress markers were modulated by the interaction of stress and diet, as well as by the bacterial composition of the gut contents. In particular, members of the genera Cetobacterium, Mycoplasma and Photobacterium are affected by both dietary changes and stress. We found evidence for the metabolic activity of bacteria and, to a lesser extent, viruses and archaea, characterised by the expression of several genes. Diet and handling stress significantly modulated the expression patterns of these genes.&nbsp;</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue95hlVTCd" id="content10-by">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Piglet Fecal Microbiome Profiling Before the Onset of Post-Weaning Diarrhea [6]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Moiz Khan Sherwani [University of Copenhagen]</strong><br><br><em>Coauthors: Mattia Pirolo, Carmen Espinosa-Gongora, Esben Østergaard Eriksen, Raphael Eisenhofer, Antton Alberdi, Luca Guardabassi</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Post-weaning diarrhea (PWD) is an economically important disease in pig production. To understand the changes in the gut microbiota of piglets affected by PWD, a longitudinal study was conducted on two Danish pig farms spanning 12 days after weaning. Clinical status of individual piglets was recorded daily, and rectal swabs were collected on seven days, yielding a total of 1,050 samples from 150 piglets. Based on clinical examination, we selected 50 piglets that developed PWD and 16 that remained healthy for 16s rRNA metabarcoding and shotgun metagenomics (250 and 113 samples, respectively). While no differences in α-diversity between these two groups prior to the onset of PWD were observed, animals that remained healthy displayed a different community structure (PERMANOVA, p=0.05) and higher abundance of amplicon sequence variants and metagenome-assembled genomes associated with beneficial bacteria such as Lachnospiraceae, Oscillospiraceae and Lactobacillaceae, which could be studied as potential probiotics. Machine learning analysis by Random Forest revealed distinctive microbial signatures related to health status that could be further exploited as biomarkers for early identification of piglets predisposed to PWD. Altogether, our study provides insights into the microbial dynamics of PWD, which may be useful for developing targeted interventions for PWD prevention and control.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue9bCk6Q3b" id="content10-c0">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Exploring the Impact of Seed Microbiota on <em>Theobroma cacao</em> L. from Evolution to Agricultural Innovation [7]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Alejandro Caro-Quintero [Universidad Nacional De Colombia]</strong><br><br><em>Coauthors: Roxana Yockteng, Deisy Toloza, José Ives Pérez Zúñiga</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The diversity and function of seed microbiota in perennial tropical plants, such as <em>Theobroma cacao</em> L., is not well understood, and it is crucial to investigate whether it is affected by agriculture and domestication. In this study, we explored the microbial endophyte diversity of <em>T. cacao</em>'s seeds and its potential role in promoting plant growth. Our research examined the taxonomic composition of endophytes in different seedling tissues germinated under sterile conditions using 16S rRNA and ITS analysis. To evaluate the effect of domestication on microbiota composition, we assessed three groups: well-established commercial varieties, recently released varieties, and landraces from Tumaco, Colombia. The analysis revealed that landraces and newly introduced varieties are dominated mainly by Pseudomonas and Pantoea genera, while commercial varieties exhibit a more extensive bacterial diversity. Fungal diversity remained consistent across all varieties. Interestingly, landraces and newly introduced varieties showed higher bacterial abundance than commercial ones, suggesting that domestication may lower endophyte abundance. Furthermore, we characterized bacterial isolates from the same seedling tissues and evaluated their plant growth-promoting capabilities. The bacterial isolates identified as Pantoea, Bacillus, and Pseudomonas had plant growth-promoting properties, such as indole production and ACC deaminase activity. We inoculated these bacteria at graft junctions and conducted experiments in greenhouse conditions, significantly increasing grafting success rates from 50 to 80% for commercial and local materials, particularly in San Andrés de Tumaco (Nariño, Colombia), where producing high-quality cocoa beans faces challenges due to low grafting success. However, the effectiveness of the bacteria varied, indicating a specific response between materials and the isolate. Our study highlights the potential of leveraging beneficial microorganisms to overcome agricultural challenges, promote healthier plant development, and contribute to regional economic growth.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue9bFpiA1j" id="content10-c5">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Study of the Honeybee Holobiont to Unravel Management and Environment-Driven Changes in Colony Resilience (WP4 BEEGUARDS, HorizonEurope Project) [8]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Melanie Parejo [University of the Basque Country]</strong><br><br><em>Coauthors: Luis Javier Chueca, Marin Kovačić, Gennaro Di Prisco, Andone Estonba, BEEGUARDS consortium, Iratxe Zarraonaindia</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The European apicultural sector suffers from average annual honeybee colony losses ranging between 20 and 50% (!) with their causes being multifactorial. The required application of zoosanitary products and feeding makes the European beekeeping sector increasingly unsustainable. A holistic approach on honeybee health and survival is urgently needed to tackle the global challenges of our most important managed pollinator. The European HorizonEurope BeeGuards project (2023-2027) aims to provide sustainable management and breeding strategies and strengthen the resilience of the European beekeeping sector in the face of climate change. Specifically, in WP4, we will implement a hologenomic framework to honeybee research by generating hologenomic sequence data (bee genomes and transcriptomes, gut metagenomes and metatranscriptomes) from field and controlled laboratory experiments from a pan-European apiary network. The hologenomic profiles will be linked to phenotypic colony observations such as colony development patterns, pathogen loads, and colony performance and survival, as well as environmental data collected from in-hive digital tools. Harnessing bees, microbes and their interaction enables BeeGuards to guide informed decision making for resilient beekeeping.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue9bEPdowA" id="content10-c4">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Determination Bacterial Transfer Route from the Microbiome of Visceral Pig Organs and its Relation to Carcass Handlers’ Hands at Slaughterhouse in Thailand [9]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Jutamat Klinsoda [Institute of Food Research and Product Development,Kasetsart University]</strong><br><br><em>Coauthors: Alongkot Boonsoongnern, Narut Thanantong, Tanyanant Kaminsonsakol, Khemmapas Treesuwan, Sudsai Trevanich, Barbara U. Metzler-Zebeli</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Gut microbiota in the porcine gastrointestinal tract has been associated with foodborne diseases. This study aimed to investigate the gut microbiota in pig feces, blood, meat cut, and visceral organs (i.e. lung, tonsil, caecum, spleen) collected at the slaughterhouse in Thailand and their relation to the microbiota found on hands of the slaughterhouse workers. At the slaughterhouse, feces, blood, tissues from 12 carcasses, and hand swab samples from 8 workers were collected. The DNA was extracted and 16S rRNA gene was sequenced. Overall, bacterial communities largely varied among sample types. Bacterial species richness (Chao1) and diversity indices (Simpson and Shannon) were significantly different between samples (p&lt;0.001). Beta-diversity analysis showed that the bacterial communities of meat cut and hand of the workers overlapped, whereas the community of tonsil clustered differently. Dominant genera across samples were Oscillospiraceae-UCG005, Acinetobacter, Anoxybacillus, Ralstonia, Escherichia_Shigella, Pseudomonas, Streptococcus, Christensenellaceae, Campylobacter, and Mycoplasma. Our findings indicate the transfer route of gut bacteria to pig organs, especially handling of meat cut, leading to contamination on the hands of workers. The study suggests that a better slaughtering process should be implemented and maintained to prevent foodborne pathogen contamination in pig products. <br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue9bGD83hh" id="content10-c7">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Unveiling Plant Holobiont Mechanisms of Root Parasite Control [10]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Lisa Martinez [Aarhus University Flakkebjerg]</strong><br><br><em>Coauthors: Enoch Narh Kudjordjie, Nina Sipari, Teemu Teeri, Mogens Nicolaisen, Mette Vestergård</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Root-knot nematodes (RKNs) are notorious plant parasites, causing considerable yield losses and damage across various crops globally. Their management remains difficult, as most nematode-targeting pesticides have been banned due to environmental and health concerns. Biocontrol methods face challenges due to the intricate interactions between the soil microbiome and the pathosystem. Prior research suggests links between host plant genomes, metabolomes and the recruitment of root microbiomes regulating RKN infectivity, notably mediated by flavonoids. We aim to identify the factors responsible for the influence of flavonoid-mediated microbial communities on nematode infectivity. Our setup dissects the effects of microbiomes from variable flavonoid regimes on RKN infection. First, we will grow <em>Arabidopsis thaliana</em> wildtypes and flavonoid mutant lines in RKN-infested soil to analyze their metabolite signature (using untargeted and targeted metabolomics) and assess their impact on microbial community dynamics and functioning (via metabarcoding and metagenomics). Secondly, we will transfer these plant-mediated microbiomes onto sterile tomato plantlets infested with RKNs to assess their effect on nematode infectivity. Finally, we will conduct survival assays of RKNs with microbial suspensions to validate the role of microorganisms in this interplay. Understanding these interactions will aid in developing sustainable control solutions using genetically suitable plants to recruit RKN-suppressive microbiomes.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue9cXRtSub" id="content10-ca">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Back to Basics – the Fundamentals of the Black Soldier Fly (<em>Hermetia illucens</em>) Microbiome [11]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Seven Nazipi Bushi [Aarhus University]</strong><br><br><em>Coauthors: Anton Gligorescu, Laurits Bundgaard Andersen, Jesper Givskov Sørensen, Andreas Schramm&nbsp;</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Black Soldier Fly larvae (BSFL) are excellent at converting low-quality industrial waste products to high value protein, such as aquaculture feed, using minimal of resources, making them a sustainable alternative to conventional agriculture. The BSFL gut microbiome plays a crucial role in digestion and bioconversion, yet the precise mechanisms governing the microbiome establishment remain unknown. This study aims to elucidate how and when BSFL acquire their gut microbiome? A transmission study was designed to map the gut microbiome throughout the entire life cycle. We collected samples from the guts of mated females, their eggs, and subsequent emerging larvae, pre-pupae, pupae, and fully matured females using 16S rRNA gene amplicon sequencing. Nine mated females were selected and transferred to individual sterile containers for egg-laying, with the eggs and neonates maintained on sterile containers to track the transmission route of potentially vertically transmitted symbionts. Meanwhile the emerging larval stages were reared on chicken feed substrate and was routinely sampled for gut and substrate material to monitor the microbiome transmission from substrate. Preliminary results suggest minimal vertical transmission with subsequent selective acquisition of the BSFL microbiome from the environment, which would allow some degree of microbiome manipulation for optimized larval production and bioconversion.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue9cXj43B2" id="content10-c9">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>A New Challenge for Aquaculture: Navigating Holobiont Resistome Dynamics for a Healthier Farmed Fish Production [12]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Marco Basili [Università di Bologna]</strong><br><br><em>Coauthors: Eleni Nikouli,&nbsp;<span style="font-size: 1rem;">Frederik D. Møller,&nbsp;</span><span style="font-size: 1rem;">Konstantinos A. Kormas,&nbsp;</span><span style="font-size: 1rem;">Gian Marco Luna,&nbsp;</span><span style="font-size: 1rem;">Grazia Marina Quero,&nbsp;</span><span style="font-size: 1rem;">Pimlapas Leekitcharoenphon,&nbsp;</span><span style="font-size: 1rem;">Frank Møller Aarestrup</span></em><div><br></div></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Antimicrobial resistance (AMR) in fish aquaculture poses a significant threat to both human health and the environment. The surge of antibiotic treatments in intense marine aquaculture practices, coupled with anthropogenic sources of pollution, are accelerating the emergence of AMR genes in farmed fish-associated bacteria, particularly in the gut microbiome. Understanding the resistome dynamics in farmed fish holobionts is thus crucial for comprehensively addressing the emergence and spread of antibiotic resistance. Here, in the light of improving the quality, safety and sustainability of farmed fish production, we examined the AMR dynamics (i.e., content and diversity) in farmed Mediterranean gilthead sea bream across its entire life cycle. Samples were collected in three Mediterranean locations, and included farmed and wild fish as well as environmental samples from aquaculture farms. Metagenomic analyses using the ResFinder database revealed a clear distinction in abundance and composition of AMR genes between farmed fish and environmental samples, both in early-stages and open-sea farming phases. Our findings highlighted beta-lactamases as the predominant class of AMR genes in adult farmed fish, with consistent profiles observed across all sites, underscoring the pervasive nature of AMR in Mediterranean aquaculture settings. Finally, results on the differences in AMR content and diversity between farmed and wild fish holobionts are provided. Our study highlights the interplay between fish host organisms, their microbiome and the environment, and emphasizes the urgency of applying comprehensive strategies to mitigate risks to human health and preserve aquatic ecosystems from AMR contamination.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue9cYJT5hf" id="content10-cc">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Deciphering Pig Gut Antimicrobial Resistance Profiles: A Comparative Study of Shotgun Metagenomic and Metatranscriptomic Approaches [13]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Judith Guitart-Matas [Institute of Agrifood Research and Technology (IRTA)]</strong><br><br><em>Coauthors: Arturo Vera-Ponce de León, Phillip B. Pope, Torgeir R. Hvidsten, Maria Ballester, Lourdes Migura-Garcia, Yuliaxis Ramayo-Caldas</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Accurate antimicrobial resistance gene (ARGs) profiling is pivotal to the establishment of targeted interventions contributing to mitigate the spread of ARGs. Profiling ARGs can be done by using two complementary sequencing methodologies, shotgun metagenomics and metatranscriptomics. While metagenomics quantifies the microbiome genetic potential, metatranscriptomics reveals the expression levels of ARGs.  However, studies using metatranscriptomics to characterize gut ARGs are scarce in pigs. Here, we aim to compare the ARG profiles derived from both approaches from 97 pig fecal samples. ARG profiles and abundances were determined using Resfinder and normalized by the sequencing depth of each sample from both metagenomics and metatranscriptomics datasets. The analysis of ARGs resulted in the identification of 173 and 106 genes by metagenomics and metatranscriptomics, respectively, with 105 genes commonly identified across both datasets. Procrustes analysis on the Aitchison distance matrix revealed strong similarity between datasets (r=0.74, p=0.001). However, it is important to note that correlations between individual ARG profiles exhibited significant variations. Among positively correlated genes with a prevalence higher than a 10% in both datasets, only 15 displayed correlations above 0.6.  In conclusion, our study showed a substantial overlap in the number of ARGs identified between metagenomics and metatranscriptomics, but also reveals large variability in the individual ARGs abundance patterns. These findings stress the importance of considering both approaches to better understand the dynamics of ARGs in gut microbial communities.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue9cZ8kIHn" id="content10-cd">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Aquaculture-Related Metagenomes for Understanding Implications of ARGs in Holobionts across OneHealth Sectors [14]</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Jacob Agerbo Rasmussen [University of Copenhagen]</strong><br><br><em>Coauthor: Morten T. Limborg</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The global challenge of antimicrobial resistance (AMR) necessitates urgent action, as highlighted by the World Health Organization's Global Action Plan (AMR-GAP). Despite commitments to combat AMR, the rise in deaths from infections caused by antibiotic-resistant bacteria underscores the critical need for improved surveillance tools and understanding of AMR. Aquaculture, crucial for meeting the increasing demand for fish and seafood sustainably, faces risks of exacerbating AMR trends due to uncontrolled antimicrobial use, especially in LMIC, a major global fish producer. The analysis of aquaculture-related metagenomes is crucial for understanding the implications of Antibiotic Resistance Genes (ARGs) in holobionts across OneHealth sectors. By examining microbial communities in aquaculture environments, insights into the dynamics of ARGs and their impact on the interconnected health of aquatic holobionts, humans, and the environment are gained. Genome-resolved metagenomics provides a powerful tool to identify factors contributing to the generation and spread of ARGs and mobile genetic elements (MGE) in aquaculture to other OneHealth sectors, facilitating better mitigation strategies. Understanding the ARGs dynamics in aquaculture is crucial for OneHealth initiatives. Here we illustrate how metagenomics can help us understand AMR by elucidating the important antibiotic resistance agents, including bacteria and mobile genetic elements (MGE) on a global scale.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue9dX97ark" id="content10-ch">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Delving into Microbial Interdependencies through Metabolic Networks Inferred from Genome-Resolved Metagenomics to Understand Broiler Performance [15]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Sofia Marcos [University of the Basque Country]</strong><br><br><em>Coauthors: Iñaki Odriozola, Jorge Langa, Andone Estonba, Antton Alberdi</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Chicken production could benefit from sustainable microbe-based solutions to increase production efficiency, but little is known about temporal dynamics of the caecal microbiome. Metabolic networks can be inferred from bacterial genomes to know what metabolites are required, used or produced by each bacterium. This information enables us to study bacterial interdependencies, which could either stabilise or destabilise the microbial community. Here, we studied genome-scale metabolic networks of 822 bacteria sequenced from the caecal content of broiler chickens during their production process. In this ongoing work, we have already identified cross-feeding examples that shape the spread of zoonotic agents. A distinct microbiota with a highly abundant Bacteroides fragilis_A is potentially producing essential metabolites for bacterial species from Campylobacterales order. We also anticipate identifying major donors of genome-reduced bacteria with several auxotrophies that became more abundant while chickens grew. These low metabolic capacity bacteria could be indicative of a more stable and mature microbiome, while appearing to be beneficial to the host. Knowing their main suppliers could help us understand how the microbial community is assembled. With these two examples we present the relevance of inferring microbial metabolic networks. <br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ufXdLxaK6y" id="content10-cp">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Genomic Prediction of Host Resistance to Sea Lice (<em>Caligus rogercresseyi</em>) in Atlantic Salmon (<em>Salmo salar</em>) Using Skin Microbiome Information [16]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Lucas Venegas&nbsp; [University of Chile]</strong><br><br><em>Coauthors: Pablo Cáceres, Sebastián Zavala, Javiera Espinoza, Jean Paul Lhorente, Nicolas Derome, José Manuel Yáñez</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Aquaculture is the fastest-growing production sector globally and an important source of food resources for the population. However, its production is affected by various diseases and parasitic infestations. One of the industry's most prevalent and harmful parasites is the sea louse (<em>Caligus rogercresseyi</em>), which causes economic losses and physical damage to fishes, and Atlantic salmon (<em>Salmo salar</em>) is a susceptible species to this parasite. Resistance to sea lice is a highly polygenic character, and the different genomic selection models proposed to date have not shown a significant improvement in the predictive values of resistance to sea lice in <em>Salmo salar</em>. Some studies have shown a more significant genetic contribution of the microbiome in the predictive models to explain the phenotypic variation of a character in production animals. This information is essential to model combining microbiome and genome, from an hologenomic perspective, which can increase the current predictive estimators for the phenotype of parasite resistance. Therefore, this study aims to evaluate the contribution of dermal microbial composition on the character of resistance to <em>Caligus rogercresseyi</em> in <em>Salmo salar </em>species, through genomic prediction models that include information from the dermal microbiome, to improve the accuracy of current predictive methods of parasite resistance.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue3NPn9OHo" id="content10-bd">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>The Response of the Tadpole Hologenome to Warming Buffers Hosts from Heat Stress [17]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Samantha Fontaine [Kent State University]</strong><br><br><em>Coauthor: Kevin Kohl</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Identifying factors that impact ectotherm physiological responses to increasing temperature is a conservation priority. We have previously shown that disrupting the tadpole gut microbiome reduces host survival under heat stress, however, the mechanisms behind these effects are unclear. Here, we compared host gene expression profiles of the tadpole gut, and their gut microbial metagenome, in natural or disrupted microbiome states, and under cool or warm conditions. Tadpoles with disrupted microbial communities exhibited a much stronger degree of host gene expression plasticity in response to heat, while the microbiome of tadpoles with more natural communities was significantly more heat sensitive. These patterns indicate that functional changes in the microbiome in response to heat may allow for a dampened host response, ultimately buffering hosts from the deleterious effects of heat stress.  Our results demonstrate how the microbiome shapes host responses to environmental stressors, which has relevance for ectotherm persistence under climate change.&nbsp;</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue95eBxZZl" id="content10-bs">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Host Phylogeny Structures the Gut Bacterial Community within Galerucella Leaf Beetles [18]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Yueqing An [Stockholm University]</strong><br><br><em>Coauthors: Mariana Braga, Sarahi Garcia, Magdalena Grudzinska-Sterno, Peter Hambäck</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Gut microbes play important roles for their hosts. Previous studies suggest that host-microbial systems can form long-term associations over evolutionary time and the dynamic changes of the intestinal system may represent major driving forces and contribute to insect dietary diversification and speciation. Our study system includes a set of six closely related leaf beetle species (<em>Galerucella spp.</em>) and our study aims to separate the roles of host phylogeny and ecology in determining the gut microbial community and to identify eventual relationship between host insects and gut bacteria. We collected adult beetles from their respective host plants and quantified their microbial community using 16S rRNA sequencing. The results showed that the gut bacteria community composition was structured by host beetle phylogeny, where more or less host-specific gut bacteria interact with the different Galerucella species. For example, the endosymbiotic bacteria Wolbachia was found almost exclusively in <em>G. nymphaea</em> and <em>G. sagittariae</em>. Diversity indicators also suggested that α- and β-diversities of gut bacteria communities varied among host beetle species. Overall, our results suggest a phylogenetically controlled co-occurrence pattern between the six closely related Galerucella beetles and their gut bacteria, indicating the potential of co-evolutionary processes occurring between hosts and their gut bacterial communities.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue4xSjTdHV" id="content10-bi">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Gut Microbiome Colonization Mediates Metabolic Division of Labor in Social Insects [19]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Tali Magory Cohen [Tel Aviv University]</strong><br><br><em>Coauthors: Levona Bodner, Sondra Turjeman, Efrat Sharon, Alisa Cohen, Sofia Bouchebti, Evgeny Tikhonov, Omry Koren, Eran Levin</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The gut microbiota is closely linked to host well-being, influencing physiology, immune function, and even social interactions. Despite its significant role, understanding the factors that determine the establishment of specific gut bacterial communities remains challenging. Here we aimed to identify factors shaping the microbiomes of nursing workers and larvae of two Vespidae species, using a cross-species common garden experimental design. By facilitating interactions between newly emerged workers (with negligible microbiota) and early-stage larvae, we examined the effects of species, trophallaxis and diet on the colonization patterns of gut bacteria. We found that the microbiome composition of adults did not significantly differ based on the identity of the nursed larvae, suggesting it primarily depended on diet. Conversely, larval microbiome composition differed more between host species, regardless of treatment. Despite ongoing trophallaxis, we found distinct microbiota profiles between life stages, as well as between the two species. Taken together, these findings suggest an evolutionary role for distinct microbiome profiles between workers and larvae, likely supporting metabolic division of labor. These results suggest that holobiont evolution may have promoted the rise of social behavior in animals.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue95hTMNTx" id="content10-bz">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Investigating Associations between Pack Management Strategies and the Intestinal Microbiota of Greenland Sled Dogs (<em>Canis lupus familiaris borealis</em>) [20]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Elsa Brenner [University Centre of the Westfjords]</strong><br><br><em>Coauthors: Pernille Klein-Ipsen, Rosalina Rotovnik, Nanna Gaun, Jorge Langa, Raphael Eisenhofer, Garazi Bideguren, Antton Alberdi, Emilie Ulrikka Andersen-Ranberg, Ostaizka Aizpurua</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The Greenland sled dog (<em>Canis lupus familiaris borealis</em>, GSD) is a working dog unique to the Greenlandic Arctic. GSDs have distinct connections to their surrounding environment: they are housed outside, drink untreated water, and fed primarily raw marine mammals and fish—a reflection of the hunted foods that Greenlandic hunters provide to their communities. Due to GSD’s connectivity to humans and the natural environment, GSDs are an excellent species for monitoring the role of diet and environment on health. This study investigates differences in the intestinal microbiota of two populations of working GSDs. We compare GSDs in Ittoqqortoormiit, who are managed as described above, to GSDs in Daneborg, who are fed dryfeed, given clean water, and managed by the military. We used genome-resolved metagenomics to characterize the diversity, composition, and function of the intestinal microbiota from 58 fecal samples, resulting in 555 metagenomic assembled genomes (MAGs). Results indicate different microbial communities between cohorts, with known pathogens more abundant in Ittoqqortoormiit. Functional analysis reveals different profiles; higher levels of alcohol, antibiotic, and xenobiotic degradation in Ittoqqortoormiit suggests anthropogenic compounds in the environment. This research contributes to QimmeqHealth, a broader effort to advance knowledge of GSD health, and diseases that impact coastal Greenland. <br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue9cZwTjK0" id="content10-ce">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Characterising Penguin Microbiomes across the Southern Ocean: Influence of Sampling Location and Host Species on Gut Microbiome Composition [21]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Katie O'Brien [University of Bath]</strong><br><br><em>Coauthors: Rachel Herman, Gemma Clucas, Bryce Robinson, John Bates, Sushma Reddy, Jacob Cooper, Michael Dunne, Claire Waluda, Phil Trathan, Vicky Hunt, Jane Younger&nbsp;</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Gut microbiome studies often lack data from wild populations, especially in avian species, which play a substantial role in ecosystem function. This study aimed to: (1) characterize the diversity and function of penguin microbiota associations across the Southern Ocean; (2) examine the factors explaining variation in microbiota composition; and (3) assess the relative contribution of host and environmental factors among species or sampling sites on microbiome composition. We characterized bacterial communities from 492 faecal samples collected from seven species at 15 breeding sites in the West Antarctic Peninsula, South Georgia, South Orkney Islands, and the Falkland Islands, by sequencing the V4 region of the bacterial 16S ribosomal RNA gene. Bacillota, Fusobacteriota, Pseudomonadota, and Campylobacterota were identified as the primary gut microbiota, showing marked variation across species and sampling sites. Sampling location was the primary driver of variation (R2=37.6%), followed by host species (R2=32.7%), indicating a combination of biological and environmental effects shaping gut microbiome composition. Gentoo penguin populations north of the Antarctic Circumpolar Current (ACC) exhibited distinctly different microbiomes to their conspecifics south of the ACC, suggesting the need for separate conservation efforts. Our findings underscore the importance of understanding the microbial diversity associated with animal populations for conservation efforts.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue95gVnVV4" id="content10-bx">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Development of Efficient Microbiota Manipulation Techniques for Reptiles [22]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Garazi Martin Bideguren [University of Copenhagen]</strong><br><br><em>Coauthors: Ostaizka Aizpurua, Carlos Cabido, Antton Alberdi</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Experimental microbiota manipulation is necessary to move beyond mere correlations and establish causal links between microbial functions and animal host responses. To date, there is limited knowledge regarding methods for manipulating gut microbial communities in non-model wild vertebrates, particularly reptiles. To address this gap, we developed an effective faecal microbiota transplantation technique to transfer the microbiota from warm-adapted to cold-adapted lizards. To develop this technique, we captured and maintained 18 cold-adapted and 9 warm-adapted lizards in captivity under their natural environmental conditions using climate chambers. After an acclimation period, animals were treated with an antibiotic cocktail to deplete their gut microbiota, followed by faecal microbiota transplant. 9 cold-adapted lizards (control group) received a faecal mixture from other cold-adapted individuals, while the other 9 individuals (treatment group) received it from warm-adapted individuals. Faecal samples were collected when animals were captured in the wild, as well as before and after the treatment to track microbiota variation until 2 weeks after transplantation. Microbiota dissimilarities were measured by means of pairwise beta diversity metrics based on Hill numbers. Our findings confirmed that both groups exhibit significantly different microbial composition in the wild, and demonstrated the suitability of our faecal microbiota transplantation procedures in transferring the gut microbiome of warm-adapted lizards to cold-adapted ones.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue4xRWqlLR" id="content10-bh">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Hologenomic Adaptation of European Whitefish - Part II [23]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Meng Zhao [University of Copenhagen]</strong><br><br><em>Coauthors: Tom Gilbert, Kim Praebel</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The European whitefish (<em>Coregonus lavaretus</em>) is a classic example of adaptive radiation, having undergone parallel morphological divergences from single founding morphs, across multiple freshwater lakes.  Investigating the pivotal role of gut microbiomes in host adaptation, we examined the potencial relationships of morphological variations in these fish and their gut microbial communities. Initial findings revealed distinct clustering of whitefish gut microbiomes from three morphs within the same lake, indicating limited gut bacteria species overlap. Extending our analysis to three trimorphic lakes and one monomorphic lake, we sequenced and processed 106 gut samples using metagenomic techniques. Consistently, principal coordination analysis reaffirmed earlier observations and intriguingly aligned with ancestral lineage hypotheses from host genome studies by Praebel’s group. Furthermore, we investigated water samples from these lakes using 16S sequencing to ascertain that the fish gut microbiome is not primarily influenced by water microbe composition. This multifaceted approach sheds light on the intricate relationship between host morphology, gut microbiomes, and environmental factors in shaping the evolution of these whitefish populations.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue9bEa143C" id="content10-c3">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Influence of Bactierial Host Ecology on Viral Community Diversity <em>In Situ </em>[24]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Caroline Sophie Winther-Have [University of Copenhagen]</strong><br><br><em>Coauthors: Jacob Agerbo Rasmussen, Thomas Sicheritz-Pontén and Morten Tønsberg Limborg</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Phages are direct predators of bacteria, shaping bacterial communities through lysis or temperate infections. Understanding what drives the host-range of phages is crucial for deciphering the microbial community dynamics and evolution within the holobiont. Elucidating virus-host interactions is challenging due to the complexity of natural systems or the limitations of<em> in vitro </em>studies. In this study, we utilised the simple system of Atlantic salmon guts, which in a healthy state is almost solely dominated by the intracellular and cell wall-lacking <em>Mycoplasma sp</em>. From a trial with a spontaneous disease outbreak causing half of the fish to develop a dysbiotic microbiota consisting of the opportunistic pathogen, a salmon-specific <em>Aliivibrio sp</em>., we explored the viral community profiles. We found that the pronounced difference between the two bacteria had a significant impact on the viral community composition. In gut systems dominated by <em>Mycoplasma sp.</em>, few to no viruses were present, whereas a highly diverse virome was present in samples dominated by <em>Aliivibrio sp.</em> bacteria. The study contributes novel insight into the significance of bacterial ecological trade-offs.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue4xTcOFr3" id="content10-bk">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Resolving Gut Microbiome Assembly Mechanisms Using Diverging Host Populations [25]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. David S. Richardson [University of East Anglia]</strong><br><br><em>Coauthors: Sarah F. Worsley, Eleanor C. Sheppard, Claudia A. Martin, Thomas J. Brown, Juan Carlos Illera</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The vertebrate gut microbiome plays an important role in host health. In nature, different populations of a host species often harbour similar, but not identical, gut microbiome communities. A key unresolved question is whether patterns of microbiome similarity across related populations ('phylosymbiosis') arise due to coevolution between host and microbial lineages, or through the repeated parallel uptake of similar microbes from the environment. Using a unique system of 13 diverging island populations of Berthelot’s pipit (<em>Anthus berthelotii</em>), we aim to determine the relative importance of these processes in shaping gut microbiome variation. Sequencing of the 16S rRNA gene has shown that gut microbiome diversity and composition differs across pipit populations. We are now combining host genomic data with shotgun metagenomic sequencing of faecal samples to understand how fine-scale gut microbiome taxonomic structure and functionality are shaped by host evolutionary history. We aim to (1) determine whether most microbes - or only specific gut microbiome lineages (e.g. pathogens, mutualists) - codiversify with their host and, (2) the extent to which microbiome taxonomic variation scales up to microbiome functional differences amongst populations. This will shed light on the processes underlying host-microbiome assembly and evolution and the possible impact of population fragmentation on host-microbe interactions.&nbsp;</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue95fAoBAa" id="content10-bu">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Senescence and Mortality and the Gut Microbiome in a Vertebrate Species Using Shotgun Metagenomics [26]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Chuen Zhang Lee [University of East Anglia]</strong><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">While the influence of the gut microbiome (GM) on human health has been extensively researched, its impact on aging and senescence in wild animal populations remains unclear. Indeed studies addressing such questions in wild population have yielded conflicting findings. Part of the problem is that acquiring the lifelong longitudinal data on individuals required to gain resolution on ageing effects is extremely challenging. This study addresses these limitations using a population of Seychelles warblers (<em>Acrocephalus sechellensis</em>) and analysing individual-based longitudinal metagenomic GM data to investigate how functional GM diversity, composition, and stability are associated with host age and senescence. Studies on humans suggest a complex interplay between the GM and host senescence, where age-related declines in physiological function appear to be linked to shifts in microbial composition and reduced GM stability. Therefore we hypothesise that GM functionality will remain stable for most of adult life but differ significantly in late life, and especially just before death, as a result of host senescence. By using a combination of metagenomics and repeated sampling of individuals, we offer a more nuanced, and potentially more accurate, analysis compared to previous metabarcoding studies. In future work, this data will be analysed alongside host genomes to better understand how inbreeding (assessed through runs of homozygosity) interacts with the GM to influence host senescence trajectories.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue9cYfpIyS" id="content10-cb">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Impact of Microbiome on Cnidarian Lifestyle: Studies in <em>Exaiptasia pallida</em> and <em>Cassiopea xamanacha </em>[27]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Federica Montesanto [University of Florida]</strong><br><br><em>Coauthors: Mark McCauley, Samuel Bedgood, Cody Miner, Bailey Steinworth, Keyla Plichon, Mark Q. Martindale, Virginia Weis, Sandra Loesgen</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Many cnidarians, including corals and anemones, engage in a symbiosis with photosynthetic dinoflagellates (family Symbiodiniaceae) that together form the trophic and structural foundation of the coral reef ecosystem. Despite the importance of these ecosystems and the threatened state of coral reef health in an era of climate change, we still have an incomplete understanding of how the partnerships are established, regulated and maintained. <em>Exaiptasia diaphana</em> and <em>Cassiopea xamanacha</em> are well-established models for exploring coral-<em>Symbiodiniaceae</em> symbiosis. Our study extends beyond algal symbionts to examine the diverse bacterial and eukaryotic communities associated with cnidarians, which have not been extensively studied. We use both marine models to investigate how the microbiome affects cnidarian-algal symbiosis and development. Specifically, we conducted experiments on aposymbiotic specimens of both species, introducing them to different species of algae, microbiome engineered algae, and photosynthetically mutated algae. Our analyses focused on symbiont acquisition, behavioural changes, and developmental indicators such as pedal laceration and asexual budding. For <em>Exaiptasia</em>, microbiome analyses across different growth stages revealed a selective adjustment in microbial and fungal communities. In <em>Cassiopea</em>, we examined how microbiome changes influence algal symbiosis, asexual budding and strobilation. The findings highlight selective and dynamic processes in choosing both, symbionts and microbial associates, which significantly influences their physiology, symbiotic relationships, development, and potentially survival.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue4xUj1FDR" id="content10-bn">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Can the Gut Microbiome Underlie Winter Adaptation in Wild Great Tits? [28]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Charli Davies [University of Jyväskylä]</strong><br><br><em>Coauthors: Antoine Stier, Andreas Nord, Suvi Ruuskanen</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Winter poses a key environmental challenge to survival in endotherms. Combination of reduced temperatures, resources, and day lengths present energetic and thermoregulatory challenges in wild populations living at high latitudes. By increasing metabolic rate through thermogenesis and mitochondrial bioenergetics, endothermic animals can adapt to decreasing temperatures by maintaining a stable body temperature, but this is energetically costly. One promising mechanism for regulating energy acquisition and heat production and thus enabling the host to adapt to rapid environmental perturbations, is the gut microbiome (GM). To see if the GM can act as a mechanism for winter adaptation, we first study longitudinal associations between winter conditions and the GM in two wild populations of great tits (<em>Parus major</em>), which experience varied environmental conditions and wild-caught individuals in captive conditions kept under either summer or winter temperatures. Secondly, to assess whether temperature-induced changes in the GM are linked to thermoregulation we associated GM composition with biomarkers of thermal physiology for both wild and captive populations. Combination of correlative and experimental methods using wild-caught individuals allows assessment of the role of GM-mediated adaptation to environmental changes, demonstrating a new route of thermal adaptation in birds.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue9bH2HdGY" id="content10-c8">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Parrotfish Microbiome and its Role in the Coral Reef Ecosystem [29]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Andrea Quagliariello [University of Padova]</strong><br><br><em>Coauthors: Marialaura Gallo Elisa Maritan, Davide Maggioni, Davide Seveso, Tomaso Patarnello, Maria Elena Martino</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Coral reef islands are a unique landforms composed by sediments derived from the surrounding coral reefs, which are particularly sensitive to climate change. Recently, parrotfishes have been identified as the main actor in the formation of such islands. Particularly, excavator parrotfish species contribute for at least &gt;80% of the new coral sediment production, which constitutes the substrate for reef islands formation. To date, how parrotfish microbial symbionts reacts to coral ingestions and their role in sediment formation remains elusive. To this end, we collected both oral and rectal microbiome samples from 8 different species of parrotfish, as well as corals and water samples at the Maldives islands. Preliminary analyses, based on 16S rRNA amplicon sequencing, show that the gut ecosystem is richer than the oral one in term of bacterial diversity and that both are poorly influenced by coral microbiome. Moreover, the gut ecosystem is characterized by a strong phylogenetic diversity between the excavator and scraper species, suggesting that the different feeding behaviour, as well as the different intake of coral in their diet shape the composition of the host microbiome.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue4xTXBNO4" id="content10-bm">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Impacts of Feralization on the Horse Hologenome [30]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Madeleine Van Well Bergström [Uppsala University]</strong><br><br><em>Coauthors: Carl-Gustaf Thulin, Gaia Resmini, Doortje Theunissen, Dominic Wright, Elin Videvall</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Feralization is a process whereby domestic animals are returned to wild conditions and is often designated as a reverse domestication event. Despite intense focus on genetic and microbial changes associated with domestication, we know little about how feralization affects animal hologenomes. Feralization can lead to changes in the morphology, physiology, behavior, and microbiota of animals, due to the process involving a sudden return of natural and sexual selection. Thus, feralization provides a unique opportunity to observe the genomic, phenotypic, and microbiota responses to selection from a known (domesticated) standpoint. In this study, we investigate the impacts of feralization on a population of feral Pottoka horses in northern Spain. We sequenced their gut microbiomes using metabarcoding and their genomes with RAD-sequencing. Variation in gut microbiome composition and diversity will be evaluated across individuals, compared with nearby domesticated horses, and in relation to their genetic information. These results will enable a better understanding of the process of feralization as microbes can more rapidly adapt to new environmental challenges. Feralization of domestic species with an extinct ancestral wildtype, such as horses, facilitates opportunities to bring back equivalents to restore ecosystem function in benefit of biodiversity.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue9bDuO1lv" id="content10-c2">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Characterizing the Microbiome of a Swedish Population of Sand Lizard (<em>Lacerta agilis</em>) Using 16S Gene Metabarcoding [31]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Charlotte Enkvist [Uppsala University]</strong><br><br><em>Coauthors: Elin Videvall, María Cortázar-Chinarro, Mette Lillie</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The sand lizard (<em>Lacerta agilis</em>) has one of the broadest distributions of any lizard species and is found across much of Eurasia, however its Swedish populations are currently threatened due to habitat loss. As we gain understanding of how the microbiome plays a key role in the fitness of the host, characterizing the microbiome is important for further conservation efforts of this and other lizard species. This study sampled individuals from an island population of sand lizards outside Gothenburg, Sweden. The samples were sequenced for the 16S gene region with the aim of characterizing the microbiome diversity and explore intraspecific variation within sex and seasonality, and also compare results from non-lethal sampling methods. The microbiome was mainly composed of taxa belonging to the phyla Actinobacteria, Firmicutes, Proteobacteria and Bacteriodota. Cloacal and fecal samples were compositionally more similar than buccal samples, and buccal samples had a higher ASV richness. Females had a significantly higher richness and were compositionally different from males in the cloacal microbiome. There was no significant difference between temporal groups. We now have sufficient knowledge of this host species’ microbiome to conduct more thorough longitudinal studies and investigate interactions between the microbiome, immune response and parasite load.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue95g5MVrj" id="content10-bv">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>The Aquatic Symbiosis Genomics (ASG) Project [32]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Michael Paulini [Wellcome Trust Sanger Institute]</strong><br><br><em>Coauthors: Genome Reference Informatics Team</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The Aquatic Symbiosis Genomics Project seeks to provide data for 500 holobiomes of more than 1000 freshwater and marine species. This includes high quality reference genomes for hosts species, as well as genomic and transcriptomic data for the holobiome, their annotation and description. The contributing community hubs are covering a wide range of research from sponges, photosymbiosis, coral symbiosis, symbiosis in single-cell eukaryotes, ciliates, cephalopods, annelids and lichens. And in addition, as a Moore Foundation funded project, all of the data is being made openly available for reuse and published in Wellcome Open Research Genome Notes. We will show the progress of the project and how it can support your genomic and transcriptomic analysis.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ufECBIJUBN" id="content10-co">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Sex-related gut microbiota profiles in geographically distant populations of the Norway lobster (<em>Nephrops norvegicus</em>) [33]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Konstantinos Kormas [University of Thessaly]</strong><br><br><em>Coauthors: Eleni Nikouli, Michele Casini, Gioacchino Bono, Elena Mente</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Currently, there is very little evidence for genetic variability of the Norway lobster (<em>Nephrops norvegicus</em>) populations along the European coastline. We hypothesised that the gut microbiomes of these populations would be characterised by a structured core microbiota profile. We tested this by investigating the gut bacterial communities of three Norway lobster populations from the eastern North Sea, (Sweden; SWE), Strait of Sicily (Italy; ITA) and North Aegean Sea (Greece; GRE) captured between Aug. and Oct. 2023. Each investigated population consisted of seven female and seven male individuals. The Aegean Sea female individuals were all carrying eggs, which were analysed separately. Statistically significant differences on males-females from the GRE and SWE populations, the GRE eggs and GRE adults (both males and females), and the total of the three populations (males+females) were found. The dominant bacterial operational taxonomic units (OTUs) were affiliated to the Firmicutes, Fusobacteriota, Bacteroidota, Desulfobacterota, Actinobacteriota (prevailed almost exclusively in eggs) and Spirocahetota phyla; several unclassified bacterial taxa occurred, especially in the GRE females. Core microbiota between the three populations consisted of 253 OTUs (10.7% of the total OTUs found in all adults). The overlap percentage of OTUs between the eggs and the GRE females and males was 24.4% and 10.1% of the total OTUs between eggs and GRE adults, respectively. This study showed that despite the genetic homogeneity of the Norway lobster populations, the differences of gut microbiota in geographically distant populations indicate a more important environmental and/or sex-based effect.&nbsp;<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue95b2VuZH" id="content10-bq">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>A Sustainable Suite of Bioinformatic Pipelines for Holo-omics [34]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Jorge Langa [University of the Basque Country UPV/EHU]</strong><br><br><em>Coauthors: Amalia Toffano, Carlotta Pietroni, Raphael Eisenhofer, Antton Alberdi</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Holo-omics, the study of the interplay between hosts and their microbiota using omic data, requires integrating data from various sources using multiple bioinformatic tools. However, this approach can be challenging due to the need to set up and use dozens of tools, understand their interoperability, and execute them in an HPC environment. To address this challenge, we developed a suite of user-friendly pipelines to conduct five common bioinformatic tasks in holo-omics: host genotyping, host transcriptome quantification, microbial metagenome assembly, microbial metagenome quantification, and microbial metatranscriptome quantification. Relying on Snakemake workflow management, these pipelines only require the path to the raw sequence files and reference (meta)genomes, and a minor set of configuration options, while software dependencies are managed automatically through Conda. We successfully applied these pipelines to the 3D'omics H2020 project, which combines host and microbiome data from chicken and swine. Our approach simplifies the setup and execution of bioinformatic tasks for hologenomics researchers, making it easier to integrate data from different sources and advance our understanding of host-microbe interactions.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue4xQCTRuU" id="content10-bf">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Three-Dimensional Simulations of Host-Microbiota and Microbiota-Microbiota Interactions [35]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Nikolai Bykov [CNAG]</strong><br><br><em>Coauthors: Bryan Wang, Zoe Horisberger, Marta Contreras, Carlotta Pietroni, Urvish Trivedi, Antton Alberdi, Marc A. Marti-Renom</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Understanding how host tissues and microbes interact is crucial for improving animal production efficiency. Conventional multi-omic approaches have limitations in revealing these interactions due to their inability to capture the spatial arrangement of relevant biological and chemical components. In the 3D'omics consortium, we aim to address this challenge by developing a new technology to generate three-dimensional data of the intestinal ecosystem and a computational framework that will enable the reconstruction of host-microbiota interactions based on this data with unprecedented spatial resolution. As a part of this framework, we introduce Gorgona, a tool for simulating the three-dimensional architecture of host tissues and bacteria based on the generated 3D'omics data. By running simulations from a random state of the environment, Gorgona reveals experimentally testable interaction patterns within the microbiota. In summary, Gorgona provides a visual and computational perspective to study the spatial distribution of microbiota in host tissues.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue4xTyFkZO" id="content10-bl">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Large-Scale Estimation of Bacteria and Archaea from Metagenomes Reveals Biome-Specific Patterns [36]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Raphael Eisenhofer [University of Copenhagen]</strong><br><br><em>Coauthors: Antton Alberdi, Ben Woodcroft</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The recovery of metagenome-assembled genomes (MAGs) from metagenomic datasets is usually incomplete, often with substantial proportions of reads remaining unaccounted for. This can be due to many factors, including appreciable levels of  non-microbial DNA in metagenomes, complexity, and insufficient sequencing depth. However, there are currently no reliable methods for pinpointing such issues in genome recovery efforts. Here we present “SingleM microbial_fraction” (SMF), a scalable algorithm that estimates the fraction of bacterial and archaeal reads in metagenomes and allows for the calculation of the Domain-Adjusted Mapping Rate (DAMR) to appraise genome recovery efforts. We benchmark SMF on simulated and real data, and demonstrate how it can be used to identify reasons for poor genome recovery. Applying SMF to ~160,000 publicly available metagenomes, we find substantial levels of variation in microbial fractions and biome-specific patterns of microbial abundance. As the adoption of metagenomic sequencing continues to increase, we foresee SMF being a valuable tool for the appraisal of genome recovery efforts.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue95gwTQxT" id="content10-bw">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>The Development of Non-Classical Intestinal Organoid Models for <em>In Vitro</em>&nbsp;Investigations into Host-Microbiota Interactions [37]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Michael Gerard Miskelly [University of Copenhagen]</strong><br><br><em>Coauthors: Kees Blijleven, Garazi Martin Bideguren, Ostaizka Aizpurua</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Laboratory animals have classically been the first choice for <em>in vivo</em> investigations as they provide invaluable insights into both physiological functions and systemic interactions between organs. However, the ethical considerations regarding laboratory animals for investigating molecular and cellular processes have become increasingly prominent. The description of murine and human intestinal organoids in 2009 and 2011 respectively has opened a new avenue for the exploitation of <em>in vitro</em> models to investigate host-microbiota interactions in the gut. Intestinal organoids provide a platform for particular experimental designs which allow us to answer specific research questions particularly with the development of new organ modelling technologies such as intestine-on-a-chip. The majority of existing research has focused on intestinal organoids derived from traditional animal models. However, there is an apparent gap in the development of intestinal organoids from diverse animal species which will allow us to answer questions from an evolutionary hologenomics perspective. The work described here aims to bridge this gap, by describing the generation of intestinal organoids from two wall lizard species<em> Podarcis muralis </em>(common wall lizard) and <em>P. liolepis </em>(Catalonian wall lizard). This is the first description of intestinal organoids from both reptilian and cold-blooded species.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue95f3eJc0" id="content10-bt">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Quantifying the Zebrafish Gut Hologenome Using Whole Tissue Nanopore Sequencing [38]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Søren Blikdal&nbsp;Hansen [University of Copenhagen]</strong><br><br><em>Coauthors: Michael L. Pepke, Morten T. Limborg</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Following the holobiont theory, the host phenotype is shaped by the host in combination with its microbiota. Hologenomics provides a practical method for studying the role of the host and the microbiota in various host phenotypes, such as the growth rate in Zebrafish. Shotgun metagenomics is a widely used approach, and sequencing efforts are often directed toward the microbiota to overcome the challenge of low microbial DNA levels. While this approach enables the assembly of high-quality Metagenome-Assembled Genomes (MAGs), it usually limit downstream analysis to relative abundances among microbes and complicates meaningful quantification of the microbial load in a sample. We employed PCR-free whole-genome sequencing of the entire gut of 108 zebrafish using Oxford Nanopore sequencing. Most reads mapped to the zebrafish genome, with the mitochondrial genome averaging 25 times higher coverage than the nuclear genome, and showing a significant negative correlation between the mitochondrial fraction and age/growth rate. Correspondingly, the median coverage of the total MAG catalog was, on average, 10 times lower than the coverage of the zebrafish genome, though it varied among samples and MAGs. In some samples the MAG coverage of <em>Aeromonas sp., Pseudomonas sp.</em>, and <em>Cetobacterium sp.</em> exceeded the coverage of the zebrafish genome, suggesting bacterial cells outnumber zebrafish cells 10-fold at times, assuming unbiased sequencing of the gut hologenome. The great variability in the ratio between hologenomic elements may reveal new aspects in our understanding of hologenomic interactions and growth in zebrafish.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-ue9bG2hS1c" id="content10-c6">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><div><strong>Ultra-High Throughput Single-Microbe Sequencing Enabled by Semi-Permeable Capsules [39]</strong></div></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Justyna Mazul [Atrandi Biosciences]</strong><strong><br></strong><br><em>Coauthors: Vaidotas Kiseliovas, Vaida Kurmauskaite, Simonas Jocys, Lorea Campos, Rapolas Zilionis</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Whole-genome sequencing helps to understand the diversity and function of unculturable microorganisms. Metagenomics is attractive for its straightforward library preparation from bulk environmental samples but only offers limited resolution into individual species. On the other hand, single-microbe sequencing offers true single-clone resolution but meaningfully addresses the high biological diversity if performed on thousands of individual cells in parallel. The Semi-Permeable Capsule (SPC) technology enables an unlimited number of processing steps on genetic material from thousands of individual isolated microbes in parallel. This study aimed to demonstrate the use of SPCs for barcoding 10,000 individual microbial genomes to obtain whole genome sequencing data of unprecedented quality. For a proof-of-concept demonstration, we processed <em>E. coli</em> and<em> B. subtilis</em> bacteria. Sequencing results showed &gt;90% genome recovery from individual cells. Next, we applied the method for environmental samples. We encapsulated microbial cells from soil, pond water and saliva samples, lysed cells at alkaline conditions, amplified their genomes, and employed a split-pool approach to add cellular barcodes. We then sequenced an aliquot of barcoded cells and evaluated technical metrics related to <em>de novo</em> assembly of SAGs. We conclude that the compartmentalization of microbial cells into SPCs allows the generation of high-quality whole-genome data at scale.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content12 cid-th8pbrSF3I" id="content12-4i">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-md-12 col-lg-10">
                <div class="mbr-section-btn align-center"><a class="btn btn-primary display-4" href="AHC_Abstracts.html#top"><span class="mobi-mbri mobi-mbri-arrow-up mbr-iconfont mbr-iconfont-btn"></span>Back to top</a></div>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="footer4 cid-sODZ7VEkgr" once="footers" id="footer4-2y">

    
    
    <div class="container">
        <div class="row mbr-white">
            <div class="col-6 col-lg-3">
                <div class="media-wrap col-md-4 col-12">
                    
                        <img src="assets/images/picture-1-596x658.png" alt="">
                    
                </div>
            </div>
            <div class="col-12 col-md-6 col-lg-3">
                <h5 class="mbr-section-subtitle mbr-fonts-style mb-2 display-7"><strong>About</strong></h5>
                <p class="mbr-text mbr-fonts-style mb-4 display-7">The Applied Hologenomics Conference 2024 received funding from the Novo Nordisk Foundation and the DNRF<br></p>
                
                
            </div>
            <div class="col-12 col-md-6 col-lg-3">
                <h5 class="mbr-section-subtitle mbr-fonts-style mb-2 display-7">
                    <strong>Contact</strong></h5>
                <ul class="list mbr-fonts-style display-7">
                    <li class="mbr-text item-wrap"><span style="font-size: 1.2rem;">CEH contact:</span></li><li class="mbr-text item-wrap"><a href="mailto:ceh@sund.ku.dk" class="text-success">ceh@sund.ku.dk</a></li>
                </ul>
            </div>
            <div class="col-12 col-md-6 col-lg-3">
                <h5 class="mbr-section-subtitle mbr-fonts-style mb-2 display-7"></h5>
                <ul class="list mbr-fonts-style display-7">
                    <li class="mbr-text item-wrap"><span style="font-size: 17.6px;"><a href="AHC_conference_policies.html" class="text-white"><strong>Code of Conduct and Conference Policies</strong></a></span></li>
                </ul>
            </div>
            
        </div>
    </div>
</section>


<script src="assets/bootstrap/js/bootstrap.bundle.min.js"></script>
  <script src="assets/parallax/jarallax.js"></script>
  <script src="assets/web/assets/cookies-alert-plugin/cookies-alert-core.js"></script>
  <script src="assets/web/assets/cookies-alert-plugin/cookies-alert-script.js"></script>
  <script src="assets/smoothscroll/smooth-scroll.js"></script>
  <script src="assets/ytplayer/index.js"></script>
  <script src="assets/dropdown/js/navbar-dropdown.js"></script>
  <script src="assets/theme/js/script.js"></script>
  
  
  
<input name="cookieData" type="hidden" data-cookie-cookiesAlertType='false' data-cookie-customDialogSelector='null' data-cookie-colorText='#424a4d' data-cookie-colorBg='rgb(234, 239, 241)' data-cookie-opacityOverlay='80' data-cookie-bgOpacity='undefined' data-cookie-textButton='Agree' data-cookie-rejectText='REJECT' data-cookie-colorButton='' data-cookie-rejectColor='' data-cookie-colorLink='#424a4d' data-cookie-underlineLink='true' data-cookie-text="We use cookies to give you the best experience. Read our <a href='CookiePolicy.html'>cookie policy</a>.">
   <div id="scrollToTop" class="scrollToTop mbr-arrow-up"><a style="text-align: center;"><i class="mbr-arrow-up-icon mbr-arrow-up-icon-cm cm-icon cm-icon-smallarrow-up"></i></a></div>
    <input name="animation" type="hidden">
  </body>
</html>