<!DOCTYPE html>
<html  >
<head>
  
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  
  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
  <link rel="shortcut icon" href="assets/images/ahc2-human-145x145.png" type="image/x-icon">
  <meta name="description" content="">
  
  
  <title>Abstracts</title>
  <link rel="stylesheet" href="assets/web/assets/mobirise-icons2/mobirise2.css">
  <link rel="stylesheet" href="assets/bootstrap/css/bootstrap.min.css">
  <link rel="stylesheet" href="assets/bootstrap/css/bootstrap-grid.min.css">
  <link rel="stylesheet" href="assets/bootstrap/css/bootstrap-reboot.min.css">
  <link rel="stylesheet" href="assets/parallax/jarallax.css">
  <link rel="stylesheet" href="assets/web/assets/gdpr-plugin/gdpr-styles.css">
  <link rel="stylesheet" href="assets/animatecss/animate.css">
  <link rel="stylesheet" href="assets/dropdown/css/style.css">
  <link rel="stylesheet" href="assets/socicon/css/styles.css">
  <link rel="stylesheet" href="assets/theme/css/style.css">
  <link rel="preload" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i&display=swap" as="style" onload="this.onload=null;this.rel='stylesheet'">
  <noscript><link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i&display=swap"></noscript>
  <link rel="preload" as="style" href="assets/mobirise/css/mbr-additional.css"><link rel="stylesheet" href="assets/mobirise/css/mbr-additional.css" type="text/css">

  
  
  
</head>
<body>
  
  <section data-bs-version="5.1" class="menu menu1 cid-sUvgvfOreC" once="menu" id="menu1-2x">
    

    <nav class="navbar navbar-dropdown navbar-fixed-top navbar-expand-lg">
        <div class="container">
            <div class="navbar-brand">
                <span class="navbar-logo">
                    <a href="index.html#top">
                        <img src="assets/images/ahc2-human-145x145.png" alt="AHC" style="height: 4.5rem;">
                    </a>
                </span>
                <span class="navbar-caption-wrap"><a class="navbar-caption text-black text-primary display-7" href="index.html">Applied HoloGenomics Conference&nbsp;<br>30th June - 2nd July 2024<br>Copenhagen, Denmark&nbsp;#AHC2024</a></span>
            </div>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbarSupportedContent" data-bs-target="#navbarSupportedContent" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
                <div class="hamburger">
                    <span></span>
                    <span></span>
                    <span></span>
                    <span></span>
                </div>
            </button>
            <div class="collapse navbar-collapse" id="navbarSupportedContent">
                <ul class="navbar-nav nav-dropdown" data-app-modern-menu="true"><li class="nav-item dropdown"><a class="nav-link link text-black dropdown-toggle display-7" href="#" data-toggle="dropdown-submenu" data-bs-toggle="dropdown" data-bs-auto-close="outside" aria-expanded="false">Conference Scope</a><div class="dropdown-menu" aria-labelledby="dropdown-undefined"><a class="text-black dropdown-item text-primary display-7" href="index.html#content4-1f">Conference Scope</a><a class="text-black dropdown-item text-primary display-7" href="index.html#gallery3-q" aria-expanded="false">Key Themes</a><a class="text-black show dropdown-item text-primary display-7" href="index.html#slider6-7w">Invited Speakers</a></div></li>
                    <li class="nav-item dropdown"><a class="nav-link link text-black text-primary dropdown-toggle show display-7" href="agenda_tba.html" data-toggle="dropdown-submenu" data-bs-toggle="dropdown" data-bs-auto-close="outside" aria-expanded="false">Programme</a><div class="dropdown-menu show" aria-labelledby="dropdown-889" data-bs-popper="none"><a class="text-black text-primary dropdown-item display-7" href="agenda_tba.html">Programm<br></a><a class="text-black text-primary dropdown-item display-7" href="sessions.html">Planned Sessions<br></a></div></li><li class="nav-item dropdown"><a class="nav-link link text-black dropdown-toggle display-7" href="#" data-toggle="dropdown-submenu" data-bs-toggle="dropdown" data-bs-auto-close="outside" aria-expanded="false">Useful Information</a><div class="dropdown-menu" aria-labelledby="dropdown-undefined"><a class="text-black dropdown-item text-primary display-7" href="index.html#content16-1j">Abstract and registration information</a><a class="text-black show dropdown-item text-primary display-7" href="index.html#content17-n">Useful Info/Costs/Accommodation etc</a><a class="text-black show dropdown-item text-primary display-7" href="index.html#map1-o">Conference Location</a><a class="text-black show dropdown-item text-primary display-7" href="index.html#clients01-9r">Acknowledgements</a><a class="text-black show dropdown-item text-primary display-7" href="AHC_conference_policies.html">Code of Conduct, Health &amp; Safety and Conference Policies</a><a class="text-black show dropdown-item text-primary display-7" href="index.html#footer4-k">Contact</a></div></li><li class="nav-item"><a class="nav-link link text-black text-primary display-7" href="news.html">News</a></li></ul>
                <div class="icons-menu">
                    <a class="iconfont-wrapper" href="https://twitter.com/AppliedHoloConf" target="_blank">
                        <span class="p-2 mbr-iconfont socicon-twitter socicon" style="color: rgb(177, 95, 98); fill: rgb(177, 95, 98);"></span>
                    </a>
                    
                    
                    
                </div>
                <div class="navbar-buttons mbr-section-btn"><a class="btn btn-white display-4" href="registration.html">
                        Registration</a></div>
            </div>
        </div>
    </nav>
</section>

<section data-bs-version="5.1" class="header9 cid-tgdLZpbp4V" id="header9-37">

    

    

    <div class="text-center container">
        <div class="row justify-content-center">
            <div class="col-md-12 col-lg-9">
                
                <h1 class="mbr-section-title mbr-fonts-style mb-3 display-2"><strong>Conference Abstracts</strong></h1>
                
                <div class="mbr-section-btn mt-3"><a class="btn btn-primary display-4" href="AHC_Abstracts.html#content4-38">Session 1 Opening</a>
                    <a class="btn btn-primary display-4" href="AHC_Abstracts.html#content4-39">Session 2 Human Health</a> <a class="btn btn-primary display-4" href="AHC_Abstracts.html#content4-3a">Session 3 Agriculture, Aquaculture <br>and Food Production</a> <a class="btn btn-primary display-4" href="AHC_Abstracts.html#content4-40">Session 4 Biodiversity &amp; Conservation</a> <a class="btn btn-primary display-4" href="AHC_Abstracts.html#content4-45">Session 5&nbsp;Technical &amp; Methodological Perspectives</a>  <a class="btn btn-primary display-4" href="assets/files/Poster.pdf">Posters</a></div>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content4 cid-th71DXZOsY mbr-parallax-background" id="content4-38">
    
    <div class="mbr-overlay" style="opacity: 0.4; background-color: rgb(184, 215, 230);">
    </div>
    <div class="container">
        <div class="row justify-content-center">
            <div class="title col-md-12 col-lg-7">
                <h3 class="mbr-section-title mbr-fonts-style align-center mb-4 display-1">
                    Session 1: Opening</h3>
                
                
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th8nWoaY3T" id="content10-4b">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>The extended human hologenome, the next hundred years</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Rob Dunn, NC State University, USA</strong><br><br><em><br></em><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Here, I introduce the term "the extended hologenome" to reflect the set of genes that an organism relies upon including a) host genes, b) symbiont genes and, in addition, c) symbiont genes housed outside of the body. I then consider the evolution of the human extended hologenome over the last million years before by concluding with a discussion of the possible future human extended hologenomes. At the moment, many futures are possible and yet such futures have been little discussed.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th8oaspeP5" id="content10-4c">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Symbiotic dialogue about hologenomics</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Seth Bordenstein, Penn State Microbiome Center, USA</strong><div><strong>Senior Scientist Henna-Maria Kailanto, IFF, Finland<br></strong><br><em><br></em><br><br></div></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">A symbiotic perspective of host-microbiota research.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th8ob93BY9" id="content10-4d">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Multi-omics techniques applied to investigate the global human microbiome</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Assistant Professor Mathilde Poyet, University of Kiel, Germany&nbsp;</strong><br><br><em><br></em><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Biodiversity is disappearing rapidly at a global scale. These widespread losses include a critical but often overlooked ecosystem: the microbial biodiversity of the human gut. A rapidly growing body of literature now shows that this complex microbial community – the gut ‘microbiome’ – is critical to metabolism, immunity, and health. Imbalances in this community and lack of exposure to beneficial microbes have been linked to diseases like inflammatory bowel disease, autoimmune diseases, and cancer. The human gut microbiome is being profoundly disrupted by the widespread adoption of industrialized diets, lifestyles, and excessive use of antibiotics. As the global population increasingly shifts to urban centers, this critical biodiversity is at risk of experiencing further perturbations and losses. We founded the Global Microbiome Conservancy (GMbC) as an international and collaborative initiative to biobank the global biodiversity of the human microbiome before it is lost to industrialized diets and lifestyles. We aim to develop it as a non-commercial research platform for the scientific community and as a long-term central repository for infinite preservation. So far, we biobanked 10,000+ bacterial strains sampled from more than 40 communities of various lifestyles worldwide. Our first investigations expand our understanding of the evolution of human microbiomes in recent human history and illuminate how industrialization perturbs our gut ecosystems.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content12 cid-th8oUVF0t9" id="content12-4e">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-md-12 col-lg-10">
                <div class="mbr-section-btn align-center"><a class="btn btn-primary display-4" href="AHC_Abstracts.html#top"><span class="mobi-mbri mobi-mbri-arrow-up mbr-iconfont mbr-iconfont-btn"></span>Back to top</a></div>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content4 cid-th71GFgWaH" id="content4-39">
    
    <div class="mbr-overlay" style="opacity: 0.7; background-color: rgb(184, 215, 230);">
    </div>
    <div class="container">
        <div class="row justify-content-center">
            <div class="title col-md-12 col-lg-10">
                <h3 class="mbr-section-title mbr-fonts-style align-center mb-4 display-1">
                    <strong>Session 2: Human Health</strong></h3>
                
                
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pcHVZF2" id="content10-3i">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Food for thought: hologenomic adaptations to amylose and lactose in the evolving human diet</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Associate Professor &amp; Team Leader Christina Warinner - Harvard University, USA,  MPI, Germany&nbsp;</strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Human diet has radically changed since the emergence of our genus approximately 2 million years ago. Over that period, gene duplications related to digestion - such as amylase copy number - have vastly expanded, and genetic variants - such as lactase persistence (LP) - are among the genomic features under highest selection in the human genome. Although the evolutionary dynamics behind some of these changes are partly known, their microbial correlates within the human microbiome are less explored. Here we discuss hologenome adaptations to two carbohydrates, amylose and lactose, and show that far from being simple, they involve a complex interplay between the host genome and microbiome. Using direct molecular evidence from the archaeological and paleoanthropological record, we demonstrate that human dietary evolution is best understood at the level of the holobiont, and that only by investigating the host genome and microbiome in tandem can human evolution be fully understood.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pGzXrte" id="content10-3j">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Evolution and function of host-microbiome systems</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Assistant Professor<em> </em>Mathieu Groussin, Kiel University, Germany</strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Concurrent with industrialization, the human gut microbiome has dramatically decreased in diversity and shifted in composition. However, to what extent transitioning from hunter-gatherer to industrialized lifestyles impacted host-microbiome interactions and host physiology is unknown. Here, we generate gut microbiome multi omics data coupled with host physiology from dozens of populations worldwide, ranging from hunter-gatherers to fully industrialized groups, and show that intestinal inflammation, humoral immune response and patterns of horizontal gene transfers (HGT) between bacteria strongly changed with industrialization. We reveal that industrialized gut microbiomes associate with elevated secretion of intestinal immunoglobulin A, despite lower levels of parasitic incidence. Furthermore, populations with hunter-gatherer lifestyles exhibit the lowest levels of intestinal inflammation. Finally, we show that gut bacteria within the microbiome of industrialized individuals exchange genes more frequently than in non-industrialized populations, potentially in response to increased environmental perturbations. Overall, our results suggest that industrialization perturbed our gut&nbsp;ecosystem and homeostasis on many levels, which could contribute to many chronic inflammation diseases.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pHqw5Xi" id="content10-3k">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>The gut microbiota and mental disease – approaching a hologenomics view</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>
                    Prof. Karsten Kristiansen, University of Copenhagen, Denmark, BGI-Shenzhen, China, Qingdao-Europe Advanced Institute for Life Sciences, Qingdao, China</strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Monogenic diseases offer clear insight into the relation between the genome and diseases, but the importance of the host genome in relation to more complex multifactorial diseases has proven more difficult to establish. During the last decade, it has become well established that the gut microbiota may play an even more important role in relation to metabolism and immune functions, and evidence has been presented that the gut microbiota may also affect behavior. However, the exact molecular mechanisms by which bacteria in the gut exert their actions remain elusive. In this lecture, I will summarize recent work from Copenhagen, Shenzhen, and Qingdao demonstrating how distinct changes in the gut microbiota are associated several multifactorial diseases, focusing on mental disorders, and how integration of different omics-technologies and the use of mouse models allow us to move from association to causality. I will discuss our work combining multi-omics technologies to approach a hologenomics view on the development of multi-factorial disorders, and I will conclude the lecture by discussing possible functional consequences and perspectives of these findings.&nbsp;&nbsp;</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pHVWCiR" id="content10-3l">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Distinctive Streptococcus clades characterize oral niches regardless of industrialization status and oral pathology</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>
                    Dr. Irina M. Velsco, MPI, Germany</strong><br><strong><br></strong><br><em>Coauthors: Christina Warinner&nbsp;</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The genus Streptococcus is highly diverse and a common member of human, animal, and fermentation microbiomes. The streptococci are grouped into at least 8 phylogenetically-supported clades, five of which, Sanguinis, Mitis, Anginosus, Salivarius, and Mutans, are found almost exclusively in the mouth.  Within the human mouth, these Streptococcus species occupy, and are often the dominant genus in, distinct oral niches. It is not known whether niche specialization is a characteristic of human biology, whether it is structured by modern, industrialized lifestyles, or the extent to which oral health status influences the clades abundances. We explored the dominant phylogenetic clades in samples from dental plaque, dental calculus, buccal mucosa, and saliva of living industrialized and non-industrialized populations across the globe, as well as in historic dental calculus. We found that clade dominance is conserved across oral sites, regardless of industrialization status, but that each oral site is distinct. Dental plaque and calculus, which represent different developmental stages of a dental biofilm, are dominated by Sanguinis streptococci, yet a minority of plaque are dominated by Mutans streptococci, while a minority of historic dental calculus samples are dominated by Anginosus streptococci. In plaque, Mutans blooms with no change in levels of  Sanguinis, while in calculus, Anginosus becomes dominant only when Sanguinis levels drop. No associations between dominant clades and oral pathology were found in either modern or historic plaque or calculus. Saliva and buccal mucosa were both dominated by Mitis group streptococci, yet non-industrialized saliva showed a higher abundance of streptococci that are not known to inhabit humans or animals. In conclusion, neither industrialization status nor oral pathology show associations with the dominant Streptococcus clades of the oral sites we examined. However, Streptococcus profiles of non-industrialized saliva may be shaped by external factors, leading to artificially high estimates of diversity in these populations.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pIsUFqM" id="content10-3m">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Why are there different microbial community types?</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr. Karoline Faust , KU Leuven, Belgium</strong><br><strong><br></strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The human gut microbiome comes in different community compositions known as enterotypes. Alternative community compositions have also been reported for other human body sites such as the vagina and the oral cavity. Since enterotypes are differentially represented in various diseases, a better understanding of what drives community types also has implications for human health. Here, I will explore different hypotheses that have been proposed to explain alternative community types and how well they are supported by gut microbial data sets, with a particular focus on human gut microbial time series. In addition, I will emphasize the value of mathematical models to explore mechanistic explanations that can then be tested in microbiome data. </p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pJ6Gh80" id="content10-3n">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Ancient oral microbiome reveals gradual adaptation of Italian prehistoric communities to Neolithic dietary shifts towards agriculture</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>
                    Research Fellow Andrea Quagliariello, University of Padova, Italy</strong><br><br><em>Coauthors: Alessandra Modi, Gabriel Innocenti, Valentina Zaro, Cecilia Conati Barbaro, Annamaria Ronchitelli, Francesco Boschin, Claudio Cavazzuti, Elena Dellù, Francesca </em>Radina<em>, Alessandra Sperduti, Luca Bondioli, Stefano Ricci, Miriam Lognoli, Maria Giovanna Belcastro, Valentina Mariotti, David Caramelli, Marta Mariotti Lippi, Emanuela Cristiani, Maria Elena Martino, Italo Maria Muntoni, Martina Lari</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The human microbiome has become a valuable source of information about host life and health. To date, little is known about how it may have evolved during key phases of human evolution, such as the transition from hunting-gathering to agriculture occurred in the Neolithic period. This event radically changed our history, promoting sedentarism, population growth and leaving selection marks on the human genome.
<br>In this frame, which was the influence of agriculture transition on human-associated microorganisms?
<br>To answer this question, we investigated the evolution of the oral microbiome during this transition, by comparing Palaeolithic hunter-gatherers oral metagenomes with that of Neolithic farmers that populated the same area in Southern Italy. Such geographical region constitutes a perfect case-study to investigate the Neolithic transition as (i) the time when the shift towards agriculture took place is known, (ii) hunter-gatherers’ and farmers’ societies shared the same environmental background, (iii) bioarchaeological data on population diet and climate condition are available.
<br>Starting from 76 dental calculi, we reconstructed the ancient oral microbiome profiles through shotgun sequencing and identified the embedded plant remains consumed during life. The integration of these results with ancient diet and environmental data allowed us to detect two key shifts in the oral microbial community. The first took place in the early phases of Neolithic and is related to the cultural transition to agriculture; and a second shift characterized by the increase of several oral pathogens occurred in the middle of the Neolithic period, coinciding with a climate shift toward aridity that affected the area. Such dry phase influenced the agricultural practices (plant used and harvest times), and probably affected the microbiome composition as a consequence.
<br>Our findings demonstrate that the introduction of agriculture, along with specific climate conditions, greatly shaped the evolution of the human oral microbiome, ultimately affecting human health.&nbsp;<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content12 cid-th8p6w2wvW" id="content12-4f">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-md-12 col-lg-10">
                <div class="mbr-section-btn align-center"><a class="btn btn-primary display-4" href="AHC_Abstracts.html#top"><span class="mobi-mbri mobi-mbri-arrow-up mbr-iconfont mbr-iconfont-btn"></span>Back to top</a></div>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content4 cid-th71HvaDWr" id="content4-3a">
    
    <div class="mbr-overlay" style="opacity: 0.7; background-color: rgb(184, 215, 230);">
    </div>
    <div class="container">
        <div class="row justify-content-center">
            <div class="title col-md-12 col-lg-12">
                <h3 class="mbr-section-title mbr-fonts-style align-center mb-4 display-1"><strong>Session 3 Agriculture, Aquaculture</strong><div><strong>and Food Production</strong></div></h3>
                
                
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pMTvpFM" id="content10-3o">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Research and Innovation for microbiomes</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Magdalena Gajdzinska, Directorate General for Research and Innovation of the <em>European</em> Commission, Brussels</strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Microbiomes are in, on and all around us, understanding what microbiomes do, what they are, and how they interact is a new scientific frontier made now reachable by rapid advances in genomics. What we know and understand so far is that the microbiome has essential impacts on our health and on the food we produce, on plants and animals and on ecosystems in general. Unravelling their complexity offers huge potential for innovation and will be a major game changer in the way we manage our planet’s resources to obtain our food and improve our health. Microbiome research and innovation has the potential to touch upon many areas, from primary production and sustainable agricultural to food production and food science, to human health and waste management. Thereafter the co-benefits spans from reduction of GHGs emissions to increased adaptation options to climate change, from reduction of risk factors for NCDs to protection of biodiversity.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pO1fCLJ" id="content10-3p">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Engineering algal microbiomes for sustainable food and feed production</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Lone Gram, DTU BioSustain, Denmark</strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Microalgae are used as feed for fish larvae, as producers of valuable compounds such as polyunsaturated fatty acids or as part of biological waste water treatment. As all other organisms, algae are colonized by a microbial community establishing a phycosphere equivalent to the plant rhizosphere. The phycosphere may harbor algal pathogenic species, but also bacteria that improve algal growth and productivity, and bacteria that subsequently, when the algae are used as live feed in aquaculture, can suppress fish pathogenic bacteria. Thus the algal microbiome can be used in a biocontrol strategy in aquaculture allowing a reduction in use of antibiotics and, hence, the risk of antibiotic resistance. Our understanding of which microorganisms in the phycosphere that are beneficial to the algae is limited. On a species level, different algal species are hosts to different microbiomes, however, it is not known if there on an algal single cells basis is a differentiation in the microbiome. This talk will introduce the phycosphere and discuss challenges in its analyses as well as presenting examples of using algal microbiome engineering as biocontrol strategy in aquaculture.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pODtnPh" id="content10-3q">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Leveraging microbiota services through hologenomic analyses to improve gut health in farmed salmon</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr. Jaelle C. Brealey NTNU, Norway</strong><br><strong><br></strong><br><strong><em>Coauthors: Miyako Kodama, Jacob A. Rasmussen, Søren B. Hansen, Luisa Santos-Bay, Martin Hansen, Even Fjære, Lene S. Myrmel, Lise Madsen, Karsten Kristiansen, Harald Sveier, Marcus Thomas Pius Gilbert, Michael D. Martin, Morten T. Limborg&nbsp;</em></strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Atlantic salmon aquaculture is one of the most profitable and technologically advanced fish production industries in the world. However, improved practices are imperative if the industry is to sustainably meet future global production demands. Ongoing industry challenges include frequent disease outbreaks, inefficient feed conversion and losses due to unpredictable variation in fish size at harvest, even in fish from the same genetic broodstock raised in identical environmental conditions. We investigated whether a hologenomic approach could identify novel pathways linking the salmon gut microbiome, gut metabolism, and host genetic variability to the observed phenotypes, focusing on fish size at harvest. Genomes, transcriptomes, epigenomes, metabolomes, and metagenomes were generated from the guts of 460 harvest-aged Norwegian salmon (gutted weights: 0.8–7.8 kg), from the same broodstocks and open-water pens. Consistent with previous studies, the salmon gut microbiota was characterised by low microbial biomass and low alpha diversity, with most individuals dominated by a species of Mycoplasma specific to salmonids. As expected, host genetic variation was low, and a GWAS found only weak evidence for a host genetic link to microbiota composition. However, we identified subtle shifts in the microbiota that were associated with fish size, including increased abundance of Mycoplasma in larger, healthier fish. Multi-omic factor analysis revealed that these fish were also characterised by shifts in gene expression and metabolic profiles, including changes in fatty acid and lipid metabolism, concordant with the higher levels of omega-3 fatty acids observed in the muscle of these larger fish. We also identified links between Mycoplasma genome functions and host gut metabolism, specifically in prenol-associated pathways of both, suggesting host-specific adaptation of Mycoplasma to the salmon gut. Our results demonstrate the value of a hologenomic approach to more fully understand how complex host–microbe interactions shape growth and health performance in farmed salmon. </p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pPdcSwl" id="content10-3r">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>A hologenomic approach to the management of infectious diseases affecting the American oyster, Crassostrae virginica</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Marta Gomez-Chiarri, University of Rhode Island, USA</strong><br><br><em>Coauthors: Jessica Coppersmith, Tejashree Modak, Erin Roberts, Rebecca Stevick, Evelyn Takyi, David R. Nelson, David C. Nelson&nbsp;<br></em><strong><br></strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Management of infectious diseases affecting aquaculture species requires the use of multidisciplinary approaches able to disentangle the complex relationships between host, pathogens and the environment in which diseases occur.  Our collaborative research focuses on the study of microbial – host interactions in the American Crassostrea virginica using experimental manipulations with probiotics and pathogens, lines of oysters with different levels of disease resistance, and integrated functional and -omics approaches targeting the oyster holobiont in a variety of environments. Our research shows that probiont - pathogen interactions are complex, involving processes like antibiosis, quorum quenching, and detoxification. While probionts lead to immune stimulation of the larval host, the bacterial pathogen Vibrio coralliilyticus leads to immunosuppression.  We have also identified potential members of the oyster microbiota associated with oyster performance and ecosystem function through examination of microbiome composition and function in a variety of environmental conditions. Resequencing analysis of oyster genomes indicates the evolutionary significance of pathogenic pressure and environmental conditions, leading to significant expansions in the number of genes in families involved in regulation of cell death.  Transcriptome analysis of lines of oysters with varying levels of resistance to viral, bacterial, and protozoan pathogens identify serine protease inhibitors and genes in cell-death pathways (apoptosis) as potentially important in disease resistance.  We are now exploring how to integrate data from both the host and the microbial transcriptomes to identify key processes and members of the holobiont that determine oyster performance.  Our research serves to elucidate the complex interactions between members of the oyster holobiont and how these interactions determine health and function of this important seafood species.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pQ6zKyn" id="content10-3s">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Microbiota-mediated protection against Flavobacterium infection in gnotobiotic zebrafish</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Postdoctoral Researchers Rebecca J. Stevick and David Pérez Pascua, Institute Pasteur, France</strong><br><br><em>Co-authors: Bianca Audrain, Emmanuel E. Adade, Arthur Lensen, Alex Valm, Jean-Marc Ghigo</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Host-associated microbiomes play a key role in protection against infection by pathogenic microorganisms. In the context of intensive aquaculture prone to disease outbreaks, the use of beneficial or probiotic bacteria has recently emerged as a desirable prophylactic alternative to the use of antibiotics. However, the selection of probiotic bacteria is still often empirical and evidence-based identification of probiotics is limited by the complexity of bacteria-host interactions. To improve the evidence-based identification of fish probiotics and their efficacy in disease prevention, the use of germ-free or fully controlled gnotobiotic hosts is a promising strategy. A robust model of infection in larval zebrafish was previously developed by the team using Flavobacterium covae, a common fresh-water fish pathogen. We have observed that F. covae kills germ-free zebrafish but not conventional (non-sterile) fish with their natural microbiota. Cultivable bacterial strains were isolated from conventional zebrafish and screened for protection against F. covae. Among them, three potential probiotic strains protect the fish when added individually to germ-free fish: Chryseobacterium sp., Nubsella sp., and Brevundimonas sp. Current efforts focus on how Chryseobacterium sp. protects against infection by analyzing its impact on zebrafish host immune response, colonization in the presence or absence of the pathogen, and screening for transposon mutants unable to protect zebrafish larvae. We have identified a single mutation in a glycosyltransferase codifying gene that leads to loss of protection against F. covae and reduced colonization of the zebrafish. Ongoing characterization of the WT and non-protecting mutant will allow us to decipher the mechanisms involved in microbiota-mediated protection in vivo. This study will inspire novel strategies for engineering next-generation probiotics against F. covae and other pathogens in aquaculture and beyond.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pQFyMok" id="content10-3t">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>What if the unit of selection in breeding is the holobiont instead of the plant?</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Assistant Researcher Dr Amaia Nogales, ISA, Universidade de Lisboa, Portugal</strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Plant breeding aims to produce genotypes with superior traits that are economically profitable to humans. Such traits include higher biomass, yield and quality, nutritional level, disease and pest resistance, abiotic stress tolerance, and easy harvesting and processing. <br><br>In conventional plant breeding an elite variety is crossed with a donor that has a desired trait. After subsequent cycles of backcrossing of the offspring to the elite variety, plant phenotyping of the resulting population is performed and a new variety, that combines the elite traits and the new desired trait, is selected. In grapevine, the high intra-varietal diversity present in ancient varieties allows to follow a different approach. The procedure involves the phenotyping of an experimental population composed of individuals that represent a wide genetic variability, and the subsequent selection of the best genotypes based on quantitative genetic methods.<br><br>For obtaining easy genetic gains in plant breeding for a particular trait, this should be mainly controlled by genetic factors, with little influence of the environment. However, this is not the case for many traits, especially for complex traits like abiotic stress tolerance or nutrient use efficiency. These are highly influenced by plant symbiotic microbial populations, which play crucial roles in plant nutrition, growth and response to the environment, and may contribute to a substantial part of the observed phenotype. Hence, including them in plant breeding procedures could improve the efficiency of the whole process – i.e. breeding at the holobiont, and not just at the plant level.<br><br>An example on grapevine breeding on enhanced P-use efficiency will be presented, where arbuscular mycorrhizal fungal communities, which contribute to P uptake in their hosts, are considered in the statistical model developed to calculate the genetic gains obtained when selecting the suitable holobionts.&nbsp;<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pRA4Vwx" id="content10-3u">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Plant microbes and bacteriophages; the smallest, most abundant entities of plant microbiomes</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>
                    Prof. Lars Hestbjerg Hansen, University of Copenhagen, Denmark</strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Plant associated microbiomes arguably play a big role in plant resilience. The complexity of the interactions between the environment, the host and the microorganisms, approach astronomical proportions, and traditional microbiomics is often insufficiently detailed to decipher this complexity. Above ground, in the plant phyllosphere, there are abundant, but poorly studied, plant/microbe-microbe interactions occurring. In the presented, I will zoom in on the major biological components of the phyllosphere and rhizosphere of plants. These include at least one set of biological entities, the bacteriophages, that are often neglected as keystone regulatory components in plant associated microbiomes or in holobionts in general. I will present both experimental and bioinformatic data, supporting a key role of bacteriophages in the plant microbiome regulation.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pS3Bb0o" id="content10-3v">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Potential drivers of plasticity and persistence of the gut microbiome</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Itzik Mizrahi, Ben-Gurion University of the Negev, Israel</strong></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Relationships between gut microbial ecosystems and their vertebrate hosts have been shown in recent years to play an essential role in the well-being and proper function of their hosts. <br>In my lecture, I will discuss some of our recent findings on vertebrate gut microbiome ecosystem stability, development, and interaction with the host.&nbsp;&nbsp;<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pSyYMLd" id="content10-3w">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Copy number variation on ABCC2-DNMBP loci impacts the diversity and functions of the gut microbiota in pigs</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Yuliaxis Ramayo Caldas, Institute of Agrifood Research and Technology, Spain</strong><br><br><em>Coauthors: Daniel Crespo-Piazuelo, Jordi Morata, Olga González-Rodríguez, Anna Castello, Antoni Dalmau, Raquel Quintanilla, Maria Ballester&nbsp;</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Genetic variation in the pig genome can modulate the composition and function of porcine gut microbial communities. Previous studies have been focused on the association between single nucleotide polymorphisms and gut microbiota composition, but little is known about the relationship between host genome structural variants and gut microbial traits. The main goal of this study was to assess the effect of porcine copy number variants (CNVs) on the diversity and functions of the pig gut microbiota. For this purpose, we used whole-genome sequencing data from 100 healthy 60-day-old Duroc pigs to undertake a comprehensive identification of CNVs followed by a genome-wide association analysis between the estimated CNV and gut bacterial diversity. Among the identified CNVs, one gain (DUP) on ABCC2-DNMBP loci was significantly associated with the richness and the Shannon α-diversity index. Compared to their diploid counterparts, the gut microbiota of DUP pigs exhibits significantly higher α-diversity (p=7.6x10-4) and richness (p=1.4x10-4). After the identification of CNV-breakpoints, the gain of copies on ABCC2-DNMBP loci was confirmed by real-time quantitative PCR (qPCR). The variation of the CNV was positively correlated with α diversity (r=0.44, p value=7.3×10-3) and richness (r=0.54, p value=6.5×10-4). In addition, functional metagenomic predictions indicated that DUP samples had a higher relative abundance of key bacterial enzymes involved in the bile acid metabolism. Altogether, our results suggest the gain of copies on ABCC2-DNMBP loci as a putative host-genetic factor for the modulation of diversity, composition, and functions of the gut microbiota in pigs. </p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7pTaw3QW" id="content10-3x">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Elucidating host genetic effects on the microbiota using zebrafish and CRISPR/Cas- a hologenomic approach</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Eiríkur Andri Thormar, University of Copenhagen, Denmark</strong><br><br><em>Coauthors: Jacob Agerbo Rasmussen, Heidi Mathiessen, Moonika Haahr Marana, Cecilie Grønlund Clausen, Martin Hansen, Miyako Kodama, Louise von Gersdorff Jørgensen, Morten Tønsberg Limborg&nbsp;<br></em><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Host genetic effects on the microbiota are an often overlooked aspect in the context of host-microbiota interactions. The field of Applied hologenomics offers a unique framework to address if, how, and to what extent host genetics contribute to the composition of the host-associated microbiota. Understanding the effects of host genetics on the associated microbiota has great potential to be applied in future biotechnological advancements in the food production industry. One can imagine selective breeding and genetic engineering of livestock to promote a microbiota that enhances a given production trait or feed specifically tailored to be beneficial for the microbiota of a specific host genotype. With the main objective being such beneficial applications, we have, with the help of genetic engineering techniques and the extremely useful zebrafish model, devised an applied hologenomic approach to directly test the effects of host genes on the composition of the host-associated microbiota. Using the CRISPR/Cas system we have created zebrafish knockout mutants for the gene coding for tyrosinase (tyr), a rate-limiting enzyme in melanogenesis, and irf8, a gene responsible for a subset of intestinal macrophages. We then compare the microbiota of the mutant cohort to that of a wild type (WT) cohort, using multi-omic approaches such as metagenomics, metatranscriptomics, metabarcoding, and metabolomics. The initial results indicate a difference between the microbiota of a mutant zebrafish cohort and the WT cohort, providing evidence for the effects of a single gene on the host microbiota. With this, we also provide a proof of principle that zebrafish and CRISPR/Cas can be used in an applied hologenomic framework to address the effects of specific host genes on relevant traits associated with the microbiota.&nbsp;&nbsp;</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7VGbIXwE" id="content10-3y">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Supplemented diets with quercetin and epicatechin enhance the growth performance of Totoaba macdonaldi related to the improvement of gut microbiota metagenomic analysis and liver transcriptome: a hologenomic approach</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>María Fernanda Barragán-Longoria, Monterrey Institute of Technology and Higher Education (ITESM), Mexico<br></strong><br><em>Coauthors: Silvia Hinojosa-Alvarez, Jesús Hernández-Pérez, Lina Natalia González-Cobián, Emilio Fabian-Ortiz, Alma Garcia-Roche and Rocio Alejandra Chavez-Santoscoy.&nbsp;</em></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Totoaba macdonaldi is an endemic, vulnerable, carnivorous fish of the Gulf of California. It is currently being cultivated in Mexico for commercial and conservation purposes. There are key aspects that need to be addressed for its conservation and exploration to be successful. One of the most difficult to overcome is to find a diet supplementation that reaches protein requirements at an acceptable price and does not compromise its overall performance; several have been tested but so far, none has worked on a long term because their side effects on the digestive system. The aim of this study is to apply an hologenomic approach on the Totoabas diet problem, evaluating the effects of supplemented diets with quercetin and epicatechin, at two doses with and without inulin, on gut microbiota and liver expression. Conjoining metagenomic and transcriptomic analysis to understand microbiota composition and liver metabolic processes, provides novel insights into the biological mechanisms involved in its response to different diet strategies, allowing the fish optimal growth for its commercialization, and more importantly, for its successful reintegration into its natural habitat. The flavonoid supplementation (FL and FH) outperformed the Inulin and Control diets; FL and FH diets caused positive changes in the microbiota diversity by reducing the presence of pathogenic species, such as V. anguillarum and E. faecalis, and increasing the presence of beneficial organisms for intestinal health, such as Lactobacillaceae. From the differential expression analysis, FH presented enriched pathways with genes involved in lipid metabolism, these results complement those obtained in the hematological analysis, where the same diet presented a significant decrease in plasma cholesterol and triglycerides levels. From this results, flavonoid supplementation might modulate the bacterial composition of the intestinal microbiota, and microbiota can influence the absorption of these compounds and their interaction on lipid metabolism expression.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content12 cid-th8p99sExk" id="content12-4g">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-md-12 col-lg-10">
                <div class="mbr-section-btn align-center"><a class="btn btn-primary display-4" href="AHC_Abstracts.html#top"><span class="mobi-mbri mobi-mbri-arrow-up mbr-iconfont mbr-iconfont-btn"></span>Back to top</a></div>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content4 cid-th808JO3Fc" id="content4-40">
    
    <div class="mbr-overlay" style="opacity: 0.7; background-color: rgb(184, 215, 230);">
    </div>
    <div class="container">
        <div class="row justify-content-center">
            <div class="title col-md-12 col-lg-12">
                <h3 class="mbr-section-title mbr-fonts-style align-center mb-4 display-1"><strong>Session 4 Biodiversity and conservation</strong></h3>
                
                
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th7VI8IXnv" id="content10-3z">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Global hologenomics to reveal the impact of wild animal-microbiota interactions on ecological and evolutionary processes</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7">Dr. Aoife Leonard, University of Copenhagen, Denmark</h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Understanding the bi-directional interactions of wild animals and their associated microorganisms represents a key advance in eco-evolutionary studies, but to date our understanding of how such interactions act in nature is limited. A global effort to jointly study the genomic and metagenomics features of a diverse array of species will facilitate a better understanding of these bi-directional animal-microbiota interactions involved in ecological and evolutionary processes in the wild<br><br>I will introduce the Earth Hologenome Initiative, which addresses biological, technical, and strategic aspects, to help overcome current limitations in wild animal-microbiota research within an ecosystem framework. By standardising sample collection and processing, and facilitating data sharing, the EHI opens a wide range of research possibilities to improve our understanding of the underlying interactions and evolutionary patterns that explain population, temporal, and geographic variation. Such a framework is key to exploring how hologenomic information can be used to improve conservation measures.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th80KQ4n0U" id="content10-41">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Metagenomic shifts in mucus, tissue and skeleton of the coral Balanophyllia europaea living along a natural CO2 gradient</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Research Fellow Giorgia Palladino, University of Bologna, Italy</strong><em><br></em><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Using the Mediterranean coral Balanophyllia europaea naturally growing along a pH gradient close to Panarea island (Italy) as a model, we explored the role of host-associated microbiomes in coral acclimatization to ocean acidification (OA). Coral samples were collected at three sites along the gradient, mimicking seawater conditions projected for 2100 under different IPCC scenarios, and mucus, soft tissue and skeleton associated microbiomes were characterized by shotgun metagenomics. According to our findings, OA induced functional rearrangements in the microbiomes genetic potential that could mitigate the sub-optimal environmental conditions at three levels: i. selection of bacteria genetically equipped with functions related to stress resistance; ii. shifts in microbial carbohydrate metabolism from energy production to maintenance of cell membranes and walls integrity; iii. gain of functions able to respond to variations in nitrogen needs at the holobiont level, such as genes devoted to organic nitrogen mobilization. We hence provided some glimpses on the functional role of the coral associated microbiome in favoring host acclimatation to OA, remarking the importance of considering the crosstalk among all the components of the holobiont to unveil how and to what extent corals will maintain their functionality under forthcoming ocean conditions.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th80LnDIzZ" id="content10-42">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>The gut microbiota affects the social network of honeybees</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Philipp Engel, University of Lausanne, Switzerland</strong><br><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">The gut microbiota influences animal neurophysiology and behavior but has not previously been documented to affect emergent group-level behaviors. Ww combined gut microbiota manipulation with automated behavioral tracking of honeybee sub-colonies to show that the microbiota increases the rate and specialization of social interactions. Microbiota colonization was associated with higher abundances of one third of metabolites detected in the brain, including several amino acids, and a subset of these metabolites were significant predictors of social interactions. Colonization also affected brain transcriptional processes related to amino acid metabolism and epigenetic modification in a brain region involved in sensory perception. These results demonstrate that the gut microbiota modulates the emergent colony social network of honeybees, likely via changes in chromatin accessibility and amino acid biosynthesis.<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th80M4KeLA" id="content10-43">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Host-microbiota responses to a series of perturbations in two case-studies</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Adam Koziol, University of Copenhagen, Denmark</strong><br><br><em>Coauthors: Iñaki Odriozola, Aoife Leonard, Raphael Eisenhofer, Anders J. Hansen, M. Thomas P. Gilbert, Antton Alberdi&nbsp;<br></em><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Continued anthropogenic change is exposing species’ to novel environmental and dietary conditions at increasing rates than ever before. A major hurdle that is being met however, is the speed at which environmental conditions are changing which require more labile mechanisms in order to respond at faster time-scales such as the gut microbiome. Moreover, as gut microbiomes responses may be highly host-specific and as such, demonstrate varying levels of plasticity and adaptive potential to different hosts. To address how hosts of different evolutionary and ecological histories may respond to novel environmental conditions we measured the host-microbial dynamics of two species -  an insectivorous-specialist - Crocidura russula (N = 29) and an omnivorous-generalist - Apodemus sylvaticus (N = 22) to a series of different environmental and dietary perturbations. Using shotgun-sequenced fecal samples collected at the conclusion of each disturbance event we observed that not only did A. sylvaticus harbor a greater diversity of gut microbiota than C. russula, but that there was also significantly more stability with lower species and functional turnover across each perturbation. Interestingly, we observed that host-species plasticity reached functional peaks at different time-points for both species. In the case of C. russula, temperature significantly modulated metabolic processes switching to a community structure which either increased or decreased the metabolic capacity during hot and cold treatments respectively which is consistent with its life-history traits. Inversely, A. sylvaticus was less affected by environmental changes and more sensitive to dietary perturbations, and was able to alter the gut composition to exploit novel food sources.  Taken together, our data suggests that environmental perturbations will affect many different species’ in different ways. Understanding how different host-associated gut microbiomes will respond under different scenarios will be important in pin-pointing which species’ may be under increased threat and offer conservation managers an important resource in their available tool-kit.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th80MGWMrn" id="content10-44">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Hologenomic analysis of hundreds of herbarium specimens reveals the genomic basis of a plant invasion</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr. Vanessa Bieker, NTNU, Norway</strong><br><br><em>Coauthors: Paul Battlay, Bent Petersen, Xin Sun, Jaelle C. Brealey, Kathryn A. Hodgins, Michael D. Martin&nbsp;<br></em><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Host genetic effects on the microbiota are an often overlooked aspect in the context of host-microbiota interactions. The field of Applied hologenomics offers a unique framework to address if, how, and to what extent host genetics contribute to the composition of the host-associated microbiota. Understanding the effects of host genetics on the associated microbiota has great potential to be applied in future biotechnological advancements in the food production industry. One can imagine selective breeding and genetic engineering of livestock to promote a microbiota that enhances a given production trait or feed specifically tailored to be beneficial for the microbiota of a specific host genotype. With the main objective being such beneficial applications, we have, with the help of genetic engineering techniques and the extremely useful zebrafish model, devised an applied hologenomic approach to directly test the effects of host genes on the composition of the host-associated microbiota. Using the CRISPR/Cas system we have created zebrafish knockout mutants for the gene coding for tyrosinase (tyr), a rate-limiting enzyme in melanogenesis, and irf8, a gene responsible for a subset of intestinal macrophages. We then compare the microbiota of the mutant cohort to that of a wild type (WT) cohort, using multi-omic approaches such as metagenomics, metatranscriptomics, metabarcoding, and metabolomics. The initial results indicate a difference between the microbiota of a mutant zebrafish cohort and the WT cohort, providing evidence for the effects of a single gene on the host microbiota. With this, we also provide a proof of principle that zebrafish and CRISPR/Cas can be used in an applied hologenomic framework to address the effects of specific host genes on relevant traits associated with the microbiota.&nbsp;&nbsp;</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content12 cid-th8pajXb16" id="content12-4h">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-md-12 col-lg-10">
                <div class="mbr-section-btn align-center"><a class="btn btn-primary display-4" href="AHC_Abstracts.html#top"><span class="mobi-mbri mobi-mbri-arrow-up mbr-iconfont mbr-iconfont-btn"></span>Back to top</a></div>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content4 cid-th84ulepkG" id="content4-45">
    
    <div class="mbr-overlay" style="opacity: 0.7; background-color: rgb(184, 215, 230);">
    </div>
    <div class="container">
        <div class="row justify-content-center">
            <div class="title col-md-12 col-lg-12">
                <h3 class="mbr-section-title mbr-fonts-style align-center mb-4 display-1"><strong>Session 5: Technical &amp; Methodological Perspectives</strong></h3>
                
                
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th84zzc6ZV" id="content10-46">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Network analysis of host-microbiome omics data</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Prof. Torgeir Hvidsten, NMBU, Norway</strong><br><em>&nbsp;<br></em><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Host genetic effects on the microbiota are an often overlooked aspect in the context of host-microbiota interactions. The field of Applied hologenomics offers a unique framework to address if, how, and to what extent host genetics contribute to the composition of the host-associated microbiota. Understanding the effects of host genetics on the associated microbiota has great potential to be applied in future biotechnological advancements in the food production industry. One can imagine selective breeding and genetic engineering of livestock to promote a microbiota that enhances a given production trait or feed specifically tailored to be beneficial for the microbiota of a specific host genotype. With the main objective being such beneficial applications, we have, with the help of genetic engineering techniques and the extremely useful zebrafish model, devised an applied hologenomic approach to directly test the effects of host genes on the composition of the host-associated microbiota. Using the CRISPR/Cas system we have created zebrafish knockout mutants for the gene coding for tyrosinase (tyr), a rate-limiting enzyme in melanogenesis, and irf8, a gene responsible for a subset of intestinal macrophages. We then compare the microbiota of the mutant cohort to that of a wild type (WT) cohort, using multi-omic approaches such as metagenomics, metatranscriptomics, metabarcoding, and metabolomics. The initial results indicate a difference between the microbiota of a mutant zebrafish cohort and the WT cohort, providing evidence for the effects of a single gene on the host microbiota. With this, we also provide a proof of principle that zebrafish and CRISPR/Cas can be used in an applied hologenomic framework to address the effects of specific host genes on relevant traits associated with the microbiota.&nbsp;&nbsp;</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th84ABhSvj" id="content10-47">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>Connecting structure and function from organs to molecules in host-microbe interaction</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Dr Manuel Liebeke- Max Planck Institute for Marine Microbiology, Germany</strong><br><br><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Our understanding of metabolic interactions between symbiotic animals and bacteria or parasitic eukaryotes that reside within their bodies is extremely limited. This gap in knowledge originates from a methodological challenge, namely to connect histological changes in host tissues induced by beneficial and parasitic (micro)organisms to the underlying metabolites. We addressed this challenge and developed chemo-histo-tomography (CHEMHIST), a culture-independent approach to connect anatomic structure and metabolic function in millimeter-sized symbiotic animals. CHEMHIST combines chemical imaging of metabolites based on mass spectrometry imaging (MSI) and microanatomy-based micro-computed X-ray tomography (micro-CT) on the same animal.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th84Bli8PJ" id="content10-48">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>New data, same challenges: Case study of DNA-methylation associated with tenacibaculosis and microbiome composition in Atlantic Salmon (<em>salmo salar</em>)</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Søren Blikdal Hansen, University of Copenhagen, Denmark</strong><br><br><em>Coauthors: Miyako Kodama, Morten Limborg&nbsp;<br></em><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Technological development has enabled the study of host-microbe interactions on a variety of “omic” levels, which allows hologenomic studies to go beyond the study of variation at the genomic level1. Epigenomic variation is an emerging “omic” level having received increasing attention in applied hologenomics due to its manipulability by environmental factors and its heritable potential2,3. However, as a young field, many epigenomic results are presented as “preliminary” and are still to pass the test of time4. Only by understanding the current state of the field and its methodologies, one can fully interpret and integrate new omic levels as epigenomics into hologenomic analyses.
<br>
<br>Based on our work with DNA-methylation analyses of HoloFish and HoloFood samples, we will present our current approach for processing large scale whole genome bisulfite sequencing datasets into the most significant variation for hologenomic integration. Focusing on our preliminary, yet significant findings of profound epigenomic variation associated with the ulcerous skin disease tenacibaculosis and microbiome composition in Atlantic salmon, we will present current breakthroughs and hurdles in the three main aspects of epigenomic analyses: i) Exploratory analysis of genome-wide methylation patterns, where the salmon clustered according to disease phenotype and microbiome composition. ii) Identification of specific methylation differences between sample groups including validation of differentially methylated regions using an independent nanopore based method. iii) Functional annotation of the identified differentially methylated regions, where several were located within regulatory regions of genes highly relevant for the disease phenotype. 
<br>
<br>Based thereon we discuss our future directions for the best integration of the young imperfect field of epigenomics in the nascent field of hologenomics.&nbsp; &nbsp; &nbsp; &nbsp;&nbsp;<br></p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content10 cid-th84BUdEqL" id="content10-49">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-12 col-md">
                <h5 class="mbr-section-title mbr-fonts-style mb-4 display-2"><strong>A Microbial Genome Atlas enables novel insights into bacteria-host interactions via functional mapping</strong></h5>
                <h6 class="mbr-section-subtitle mbr-fonts-style mb-4 display-7"><strong>Arturo Vera-Ponce de León, NMBU, Norway</strong><br><strong><br></strong><em>Coauthors: Sabina Leanti La Rosa, Jacob Agerbo Rasmussen, Knut Rudi, Jon Olav Vik, Christian René Karlsen, Morten Tønsberg Limborg, Simen Rød Sandve, Phillip B. Pope&nbsp;<br></em><br><br></h6>
            </div>
            <div class="col-md-12 col-lg-6">
                <p class="mbr-text mbr-fonts-style display-7">Our knowledge about the salmon (Salmo salar) gut microbiota is limited, despite the importance of salmonid aquaculture as one of the most expanding food-production sectors worldwide. This hampers the study of functional interactions between host and bacteria, and the development of dietary strategies aimed at the manipulation of the gut microbiota to improve salmon welfare. Furthermore, the lack of salmon gut-derived microbial genome data makes it difficult to interpret the functional omics dataset, as much of the data do not match the available reference bacterial genomes. Combining published and unpublished data, here we present the first version of the Salmon Microbial Genome Atlas (SMGA), originating from fish reared both in fresh water and saltwater. The SMGA consists of 96 high-quality bacterial genomes recovered by both metagenomics (n=18) and culture isolation (“culturomics”; n=78), these last were recovered as complete circular chromosomes, using Illumina and long-reads Nanopore sequencing. Bacterial genomes were taxonomically assigned into 11 different genera. Genomes in the SMGA encode genes for carbohydrate-active enzymes (CAZymes) and short-chain fatty acid production, indicating that some of these bacteria can access glycans derived from both the diet or host gut epithelium and produce beneficial metabolites. When using the SMGA as a database to map three different meta-transcriptomic datasets, we could map 12 % of the reads, demonstrating the SMGA’s versatility as a tool to facilitate exploration into active bacterial populations in the salmon gut. These results showcase the SMGA as a useful resource and open new possibilities for functional studies of the salmon gut microbiota.</p>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="content12 cid-th8pbrSF3I" id="content12-4i">
    
    <div class="container">
        <div class="row justify-content-center">
            <div class="col-md-12 col-lg-10">
                <div class="mbr-section-btn align-center"><a class="btn btn-primary display-4" href="AHC_Abstracts.html#top"><span class="mobi-mbri mobi-mbri-arrow-up mbr-iconfont mbr-iconfont-btn"></span>Back to top</a></div>
            </div>
        </div>
    </div>
</section>

<section data-bs-version="5.1" class="footer4 cid-sODZ7VEkgr" once="footers" id="footer4-2y">

    
    
    <div class="container">
        <div class="row mbr-white">
            <div class="col-6 col-lg-3">
                <div class="media-wrap col-md-8 col-12">
                    
                        <img src="assets/images/social-337x176.png" alt="">
                    
                </div>
            </div>
            <div class="col-12 col-md-6 col-lg-3">
                <h5 class="mbr-section-subtitle mbr-fonts-style mb-2 display-7"><strong>About</strong></h5>
                <p class="mbr-text mbr-fonts-style mb-4 display-7">The Applied HoloGenomics Conference received funding from the NovoNordisk Foundation<br></p>
                
                
            </div>
            <div class="col-12 col-md-6 col-lg-3">
                <h5 class="mbr-section-subtitle mbr-fonts-style mb-2 display-7">
                    <strong>Contact</strong></h5>
                <ul class="list mbr-fonts-style display-7">
                    <li class="mbr-text item-wrap"><span style="font-size: 1.2rem;">CEH contact:</span></li><li class="mbr-text item-wrap"><a href="mailto:ceh@sund.ku.dk" class="text-success">ceh@sund.ku.dk</a></li>
                </ul>
            </div>
            <div class="col-12 col-md-6 col-lg-3">
                <h5 class="mbr-section-subtitle mbr-fonts-style mb-2 display-7"></h5>
                <ul class="list mbr-fonts-style display-7">
                    <li class="mbr-text item-wrap"><span style="font-size: 17.6px;"><a href="AHC_conference_policies.html" class="text-white"><strong>Code of Conduct and Conference Policies</strong></a></span></li>
                </ul>
            </div>
            
        </div>
    </div>
</section>


<script src="assets/bootstrap/js/bootstrap.bundle.min.js"></script>
  <script src="assets/parallax/jarallax.js"></script>
  <script src="assets/web/assets/cookies-alert-plugin/cookies-alert-core.js"></script>
  <script src="assets/web/assets/cookies-alert-plugin/cookies-alert-script.js"></script>
  <script src="assets/smoothscroll/smooth-scroll.js"></script>
  <script src="assets/ytplayer/index.js"></script>
  <script src="assets/dropdown/js/navbar-dropdown.js"></script>
  <script src="assets/theme/js/script.js"></script>
  
  
  
<input name="cookieData" type="hidden" data-cookie-cookiesAlertType='false' data-cookie-customDialogSelector='null' data-cookie-colorText='#424a4d' data-cookie-colorBg='rgb(234, 239, 241)' data-cookie-opacityOverlay='80' data-cookie-bgOpacity='undefined' data-cookie-textButton='Agree' data-cookie-rejectText='REJECT' data-cookie-colorButton='' data-cookie-rejectColor='' data-cookie-colorLink='#424a4d' data-cookie-underlineLink='true' data-cookie-text="We use cookies to give you the best experience. Read our <a href='CookiePolicy.html'>cookie policy</a>.">
   <div id="scrollToTop" class="scrollToTop mbr-arrow-up"><a style="text-align: center;"><i class="mbr-arrow-up-icon mbr-arrow-up-icon-cm cm-icon cm-icon-smallarrow-up"></i></a></div>
    <input name="animation" type="hidden">
  </body>
</html>